<!DOCTYPE html><html><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width"/><link rel="icon" href="/commons-dws-public/favicon.ico"/><title>Strain</title><meta name="robots" content="index,follow"/><meta name="googlebot" content="index,follow"/><meta name="description" content="Personal knowledge space"/><meta property="og:title" content="Strain"/><meta property="og:description" content="Personal knowledge space"/><meta property="og:url" content="https://commons-research.github.io/commons-dws-public/notes/a0iu9pk5ktxoo6h7y7gmm6e/"/><meta property="og:type" content="article"/><meta property="article:published_time" content="10/24/2024"/><meta property="article:modified_time" content="11/18/2024"/><link rel="canonical" href="https://commons-research.github.io/commons-dws-public/notes/a0iu9pk5ktxoo6h7y7gmm6e/"/><meta name="next-head-count" content="14"/><link rel="preload" href="/commons-dws-public/_next/static/css/47f58a23998994fc.css" as="style"/><link rel="stylesheet" href="/commons-dws-public/_next/static/css/47f58a23998994fc.css" data-n-g=""/><noscript data-n-css=""></noscript><script defer="" nomodule="" src="/commons-dws-public/_next/static/chunks/polyfills-c67a75d1b6f99dc8.js"></script><script src="/commons-dws-public/_next/static/chunks/webpack-18f1eb2dd0aac0df.js" defer=""></script><script src="/commons-dws-public/_next/static/chunks/framework-28c999baf2863c3d.js" defer=""></script><script src="/commons-dws-public/_next/static/chunks/main-1bd061c8c8d18148.js" defer=""></script><script src="/commons-dws-public/_next/static/chunks/pages/_app-8bc9d81129a787a1.js" defer=""></script><script src="/commons-dws-public/_next/static/chunks/935-4dee79e80b8641c6.js" defer=""></script><script src="/commons-dws-public/_next/static/chunks/6-50972def09142ee2.js" defer=""></script><script src="/commons-dws-public/_next/static/chunks/pages/notes/%5Bid%5D-78d472fa3b924116.js" defer=""></script><script src="/commons-dws-public/_next/static/KWnqrj5vUmxn1r58uXWxO/_buildManifest.js" defer=""></script><script src="/commons-dws-public/_next/static/KWnqrj5vUmxn1r58uXWxO/_ssgManifest.js" defer=""></script></head><body><div id="__next" data-reactroot=""><section class="ant-layout" style="width:100%;min-height:100%"><header class="ant-layout-header" style="position:fixed;isolation:isolate;z-index:1;width:100%;border-bottom:1px solid #d4dadf;height:64px;padding:0 24px 0 2px"><div class="ant-row ant-row-center" style="max-width:992px;justify-content:space-between;margin:0 auto"><div style="display:flex" class="ant-col ant-col-xs-20 ant-col-sm-4"></div><div class="ant-col gutter-row ant-col-xs-0 ant-col-sm-20 ant-col-md-20 ant-col-lg-19"><div class="ant-select ant-select-lg ant-select-auto-complete ant-select-single ant-select-allow-clear ant-select-show-search" style="width:100%"><div class="ant-select-selector"><span class="ant-select-selection-search"><input type="search" autoComplete="off" class="ant-select-selection-search-input" role="combobox" aria-haspopup="listbox" aria-owns="undefined_list" aria-autocomplete="list" aria-controls="undefined_list" aria-activedescendant="undefined_list_0" value=""/></span><span class="ant-select-selection-placeholder">For full text search please use the &#x27;?&#x27; prefix. e.g. ? Onboarding</span></div></div></div><div style="display:none;align-items:center;justify-content:center" class="ant-col ant-col-xs-4 ant-col-sm-4 ant-col-md-0 ant-col-lg-0"><span role="img" aria-label="menu" style="font-size:24px" tabindex="-1" class="anticon anticon-menu"><svg viewBox="64 64 896 896" focusable="false" data-icon="menu" width="1em" height="1em" fill="currentColor" aria-hidden="true"><path d="M904 160H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0 624H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0-312H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8z"></path></svg></span></div></div></header><section class="ant-layout" style="margin-top:64px;display:flex;flex-direction:row"><div class="site-layout-sidebar" style="flex:0 0 auto;width:calc(max((100% - 992px) / 2, 0px) + 200px);min-width:200px;padding-left:calc((100% - 992px) / 2)"><aside class="ant-layout-sider ant-layout-sider-dark" style="position:fixed;overflow:auto;height:calc(100vh - 64px);background-color:transparent;flex:0 0 200px;max-width:200px;min-width:200px;width:200px"><div class="ant-layout-sider-children"></div></aside></div><main class="ant-layout-content side-layout-main" style="max-width:1200px;min-width:0;display:block"><div style="padding:0 24px"><div class="main-content" role="main"><div class="ant-row"><div class="ant-col ant-col-24"><div class="ant-row" style="margin-left:-10px;margin-right:-10px"><div style="padding-left:10px;padding-right:10px" class="ant-col ant-col-xs-24 ant-col-md-18"><div><h1 id="strain"><a aria-hidden="true" class="anchor-heading icon-link" href="#strain"></a>Strain</h1>
<p>The followoing strains have been identified through their 16RNA sequencing data.</p>
<h2 id="streptomyces-lusitanus"><a aria-hidden="true" class="anchor-heading icon-link" href="#streptomyces-lusitanus"></a>Streptomyces lusitanus</h2>
<p>WD query for compounds <a href="https://w.wiki/BfHS">https://w.wiki/BfHS</a></p>
<p>30 compound isolated <a href="https://w.wiki/BfHK">https://w.wiki/BfHK</a></p>
<p><a title="Private" href="https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html" target="_blank" class="private">#wd (Private)</a> </p>
<p>More info in the related note <a href="/commons-dws-public/notes/kvtssreflme8xid39zy3t0n">Streptomyces lusitanus</a></p>
<h2 id="streptomyces-xanthophaeus"><a aria-hidden="true" class="anchor-heading icon-link" href="#streptomyces-xanthophaeus"></a>Streptomyces xanthophaeus</h2>
<p>WD query for compounds <a href="https://w.wiki/BfHN">https://w.wiki/BfHN</a>
<a href="https://scholia.toolforge.org/taxon/Q26293525#metabolome">https://scholia.toolforge.org/taxon/Q26293525#metabolome</a></p>
<h2 id="paecilomyces-lilacinus"><a aria-hidden="true" class="anchor-heading icon-link" href="#paecilomyces-lilacinus"></a>Paecilomyces lilacinus</h2>
<p>WD query for compounds <a href="https://w.wiki/BfHT">https://w.wiki/BfHT</a>
Not to be worked on.
Human pathogen.</p>
<h2 id="16s-dna-sequences"><a aria-hidden="true" class="anchor-heading icon-link" href="#16s-dna-sequences"></a>16S DNA sequences</h2>
<blockquote>
<p>S3
AAGTCGAACGATGAACCACTTCGGTGGGGATTAGTGGCGAACGGGTGAGTAACACGTGGGCAATCTGCCCTGCACTCTGGGACAAGCCCTGGAAACGGGGTCTAATACCGGATACTGATCGCCTTGGGCATCCTTGGTGATCGAAAGCTCCGGCGGTGCAGGATGAGCCCGCGGCCTATCAGCTTGTTGGTGAGGTAATGGCTCACCAAGGCGACGACGGGTAGCCGGCCTGAGAGGGCGACCGGCCACACTGGGACTGAGACACGGCCCAGACTCCTACGGGAGGCAGCAGTGGGGAATATTGCACAATGGGCGAAAGCCTGATGCAGCGACGCCGCGTGAGGGATGACGGCCTTCGGGTTGTAAACCTCTTTCAGCAGGGAAGAAGCGAAAGTGACGGTACCTGCAGAAGAAGCGCCGGCTAACTACGTGCCAGCAGCCGCGGTAATACGTAGGGCGCGAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGCTTGTCGCGTCGGTTGTGAAAGCCCGGGGCTTAACCCCGGGTCTGCAGTCGATACGGGCAGGCTAGAGTTCGGTAGGGGAGATCGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGATCTCTGGGCCGATACTGACGCTGAGGAGCGAAAGCGTGGGGAGCGAACAGGATTAGATACCCTGGTAGTCCACGCCGTAAACGGTGGGCACTAGGTGTGGGCGACATTCCACGTCGTCCGTGCCGCAGCTAACGCATTAAGTGCCCCGCCTGGGGAGTACGGCCGCAAGGCTAAAACTCAAAGGAATTGACGGGGGCCCGCACAAGCGGCGGAGCATGTGGCTTAATTCGACGCAACGCGAAGAACCTTACCAAGGCTTGACATACACCGGAAACGCCCGGAGATGGGCGCCCCTTGTGGTCGGAGTACAGGTGGTGCATGGCTGTCGTCAGCTCGTGTCGTGAGATGTTGGGTTAAGTCCCGCAACGAGCGCACCCTTGTCCCGTGTTGCCAGCAGGCCCTTGGGGTGCTGGGAATCACGGGAGACCGCCGGGGTCAATCGGAGGAAGGGGGG
S6
ATGCAGTCGAACGATGAAGCCCTTCGGGGTGGATTAGTGGCGAACGGGTGAGTAACACGTGGGCAATCTGCCCTTCACTCTGGGACAAGCCCTGGAAACGGGGTCTAATACCGGATACCACTCCTGCCTGCATGGGCGGGGGTTGAAAGCTCCGGCGGTGAAGGATGAGCCCGCGGCCTATCAGCTTGTTGGTGGGGTAATGGCCCACCAAGGCGACGACGGGTAGCCGGCCTGAGAGGGCGACCGGCCACACTGGGACTGAGACACGGCCCAGACTCCTACGGGAGGCAGCAGTGGGGAATATTGCACAATGGGCGAAAGCCTGATGCAGCGACGCCGCGTGAGGGATGACGGCCTTCGGGTTGTAAACCTCTTTCAGCAGGGAAGAAGCGAAAGTGACGGTACCTGCAGAAGAAGCGCCGGCTAACTACGTGCCAGCAGCCGCGGTAATACGTAGGGCGCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGCTTGTCACGTCGGATGTGAAAGCCCGAGGCTTAACCTCGGGTCTGCATTCGATACGGGCTAGCTAGAGTGTGGTAGGGGAGATCGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGATCTCTGGGCCATTACTGACGCTGAGGAGCGAAAGCGTGGGGAGCGAACAGGATTAGATACCCTGGTAGTCCACGCCGTAAACGTTGGGAACTAGGTGTTGGCGACATTCCACGTCGTCGGTGCCGCAGCTAACGCATTAAGTTCCCCGCCTGGGGAGTACGGCCGCAAGGCTAAAACTCAAAGGAATTGACGGGGGCCCGCACAAGCGGCGGAGCATGTGGCTTAATTCGACGCAACGCGAAGAACCTTACCAAGGCTTGACATATACCGGAAAGCATTAGAGATAGTGCCCCCCTTGTGGTCGGTATACAGGTGGTGCATGGCTGTCGTCAGCTCGTGTCGTGAGATGTTGGGTTAAGTCCCGCAACGAGCGCAACCCTTGTCCTGTGTTGCCAGCATGCCCTTCGGGGTGATGGGGACTCACAGGAGACCGCCGGGGTCAACTCGGAGGAGGTGGGGACGACGTCAGTCATCATGCCCCTTAG</p>
<h2 id="its-fungi-sequences"><a aria-hidden="true" class="anchor-heading icon-link" href="#its-fungi-sequences"></a>ITS Fungi sequences</h2>
<p>CH6
AGCGGAGGGATCATTACCGAGTTATACAACTCCCAAACCCACTGTGAACCTTACCTCAGTTGCCTCGGCGGGAACGCCCCGGCCGCCTGCCCCCGCGCCGGCGCCGGACCCAGGCGCCCGCCGCAGGGACCCCAAACTCTCTTGCATTACGCCCAGCGGGCGGAATTTCTTCTCTGAGTTGCACAAGCAAAAACAAATGAATCAAAACTTTCAACAACGGATCTCTTGGTTCTGGCATCGATGAAGAACGCAGCGAAATGCGATAAGTAATGTGAATTGCAGAATTCAGTGAATCATCGAATCTTTGAACGCACATTGCGCCCGCCAGCATTCTGGCGGGCATGCCTGTTCGAGCGTCATTTCAACCCTCGAGCCCCCCCCGGGGGCCTCGGTGTTGGGGGACGGCACACCAGCCGCCCCCGAAATGCAGTGGCGACCCCGCCGCAGCCTCCCCTGCGTAGTAGCACACACCTCGCACCGGAGCGCGGAGGCGGTCACGCCGTAAAACGCCCAACTTTCTTAGAGTTGACCTCGGATCAGGTAGGAATACCCGCTGAACTTAAGCATATCAATAAGCGGAGGAA</p>
</blockquote>
<h2 id="dna-protocol-extraction"><a aria-hidden="true" class="anchor-heading icon-link" href="#dna-protocol-extraction"></a>DNA Protocol extraction</h2>
<p>PCR PROTOCOL FOR Actinomycetes
DNA extraction (heat shock method):</p>
<ol>
<li>Take bacterial mycelium in a clean 1.5 mL Eppendorf tube.</li>
<li>Grind the cells using a clean tip and suspend them thoroughly in 1 mL of distilled water. </li>
<li>Heat the samples at 95Â°C in the heating block for 20 min.</li>
<li>Transfer the tube (suddenly) to ice for 10 min. </li>
<li>Re-heat the sample in the heating block at 95Â°C for 10 min.</li>
<li>Store DNA at -20Â°C.</li>
</ol>
<p>16S rDNA Primers:
â¢	27F and 1492R
27F: 5â-AGAGTTTGATCCTGGCTCAG-3â
1492R: 5â-CGGTTACCTTGTTACGACTT-3â
(A)	 PCR amplification in 25 Î¼l of reaction (Accustart II):
â¢	5 Î¼L of genomic DNA
â¢	0.5 Î¼L of primer F (25 Î¼M)
â¢	0.5 Î¼L of primer R (25 Î¼M)
â¢	12.5 Î¼L of Accustart II PCR toughmix (2x)
â¢	0.5 Î¼L of gel track loading Dye (50X)
â¢	6 Î¼L of water</p>
<p>(B)	PCR amplification in 25 Î¼l of reaction (Solifast):
â¢	5 Î¼L of genomic DNA
â¢	0.5 Î¼L of primer F (25 Î¼M)
â¢	0.5 Î¼L of primer R (25 Î¼M)
â¢	5 Î¼L of Solifast (5x)
â¢	14 Î¼L of water</p>
<p>Amplification conditions:
â¢	Initial denaturation at 94Â°C for 3 min
â¢	Followed by 35 cycles of</p>
<ol>
<li>a denaturation step at 94Â°C for 30 s,</li>
<li>an annealing step at 56Â°C for 30 s, and</li>
<li>an extension step at 72Â°C for 1 min 40 s,
â¢	Followed by a final extension step at 72Â°C for 10 min.</li>
</ol>
<p>Gel electrophoresis:</p>
<ol>
<li>Prepare 1% agarose: 1g agarose in 100 mL of 1X TAE buffer (40 mol/L Tris-acetate, and 1 mM EDTA, pH 8.0). (0.6 g in 60 mL for small trays)</li>
<li>Stain the gel with 6 Î¼l of GreenSafe solution (3 for 60 mL).</li>
<li>Immerse the gel in 1X TAE (pH 8.0) buffer.</li>
<li>Load the prepared PCR products into the wells. </li>
<li> Use a 1 Kbp-DNA ladder as a molecular weight marker. </li>
<li>Pass 100 volts of electric current through the gel for 30 min.</li>
<li>Remove and visualize the gel, in a Trans illumination cabinet.</li>
<li>Capture the image using a gel documentation system. </li>
</ol>
<p>â¢	Appearance of the target band specified for the primer set on the agarose gel is considered as a positive amplification product.</p>
<h2 id="strains-picture"><a aria-hidden="true" class="anchor-heading icon-link" href="#strains-picture"></a>Strains picture</h2>
<p>CH6
<img src="/commons-dws-public/photo_57@22-10-2024_09-06-59_thumb.jpg" alt="alt text">
<img src="/commons-dws-public/photo_92@22-10-2024_09-13-16_thumb.jpg" alt="alt text">
<img src="/commons-dws-public/photo_71@22-10-2024_09-07-00_thumb.jpg" alt="alt text"></p>
<p>S3</p>
<p><img src="/commons-dws-public/photo_97@22-10-2024_09-13-17_thumb.jpg" alt="alt text"></p>
<p>S6
<img src="/commons-dws-public/photo_53@22-10-2024_09-06-59.jpg" alt="alt text"></p>
<p>CH4
<img src="/commons-dws-public/photo_86@22-10-2024_09-13-09_thumb.jpg" alt="alt text">
<img src="/commons-dws-public/photo_63@22-10-2024_09-07-00_thumb.jpg" alt="alt text">
<img src="/commons-dws-public/photo_108@22-10-2024_09-13-17.jpg" alt="alt text">
V1
<img src="/commons-dws-public/photo_59@22-10-2024_09-06-59_thumb.jpg" alt="alt text"></p>
<h2 id="to-do-next-week"><a aria-hidden="true" class="anchor-heading icon-link" href="#to-do-next-week"></a>To do next week</h2>
<p>1- Saturday culture bacteria
1- run LC-MS for vials soil
2- ITS extraction of CH4, CH2, V1</p>
<h2 id="saturday"><a aria-hidden="true" class="anchor-heading icon-link" href="#saturday"></a>Saturday</h2>
<p>Culture CH2 and CH4 </p>
<h2 id="fungal-dna-extraction-protocol-without-kit"><a aria-hidden="true" class="anchor-heading icon-link" href="#fungal-dna-extraction-protocol-without-kit"></a>Fungal DNA extraction protocol without kit</h2>
<ul>
<li>Protocol 1
All fungal isolates were treated with 70% ethanol and heat inactivated (100Â°C for 30 min). A bead beating step was added to assist with mechanical lysis of the fungal cell wall. A viability study showed that these steps resulted in no growth of a variety of diverse clinically relevant fungi, including both yeasts and molds (Supplemental Table S2). For all samples, the Qiagen (Valencia, CA, USA) EZ1 Blood and Tissue Kit as well as the EZ1 Advanced XL instrument were used (following manufacturerâs instructions) to extract genomic DNA from fungal isolates. Post extraction, DNA was quantified using Qubit 1Ã double-stranded DNA HS assay with the Qubit 3.0 Fluorometer (Thermo Fisher Scientific, Waltham, MA, USA). DNA quantities greater than 0.04 ng/ÂµL were considered acceptable. PCR grade water was used as a negative extraction and sequencing control. A reference strain of Candida tropicalis (CDC AR Bank 3045) was used as a sequencing positive control for each run [ doi: 10.3390/jof9020183].
Je rÃ©sume:</li>
</ul>
<ol>
<li>
<p>Inactivation des isolats : Tous les isolats fongiques ont Ã©tÃ© traitÃ©s avec de l'Ã©thanol Ã  70 % et chauffÃ©s Ã  100Â°C pendant 30 minutes pour Ãªtre inactivÃ©s. Ce traitement par la chaleur et l'alcool Ã©limine la viabilitÃ© des champignons, empÃªchant leur croissance, ce qui est confirmÃ© par une Ã©tude montrant l'absence de croissance aprÃ¨s ces Ã©tapes, mÃªme pour des champignons cliniquement importants (levures et moisissures).</p>
</li>
<li>
<p>Lyse mÃ©canique : Une Ã©tape de broyage avec des billes ("bead beating") est ajoutÃ©e pour aider Ã  briser la paroi cellulaire des champignons, qui est trÃ¨s rÃ©sistante. Cette Ã©tape facilite la libÃ©ration de l'ADN contenu Ã  l'intÃ©rieur des cellules fongiques.</p>
</li>
<li>
<p>Extraction de l'ADN : Le kit Qiagen EZ1 Blood and Tissue et lâinstrument EZ1 Advanced XL ont Ã©tÃ© utilisÃ©s pour extraire l'ADN gÃ©nomique des isolats, en suivant les instructions du fabricant.</p>
</li>
<li>
<p>Quantification de l'ADN : AprÃ¨s l'extraction, la quantitÃ© d'ADN est mesurÃ©e avec le test Qubit 1Ã double-brin dâADN HS et le fluoromÃ¨tre Qubit 3.0. Seules les quantitÃ©s supÃ©rieures Ã  0,04 ng/ÂµL sont considÃ©rÃ©es comme acceptables pour les analyses suivantes.</p>
</li>
</ol>
<p>5.ContrÃ´les de qualitÃ© : De l'eau de qualitÃ© PCR est utilisÃ©e comme contrÃ´le nÃ©gatif pour s'assurer qu'aucune contamination n'est introduite pendant l'extraction et le sÃ©quenÃ§age. Une souche de rÃ©fÃ©rence de Candida tropicalis (CDC AR Bank 3045) sert de contrÃ´le positif pour vÃ©rifier la qualitÃ© et la prÃ©cision de chaque sÃ©quenÃ§age.</p>
<h2 id="culture-v1-in-tsb-for-tomorrow"><a aria-hidden="true" class="anchor-heading icon-link" href="#culture-v1-in-tsb-for-tomorrow"></a>Culture V1 in TSB for tomorrow</h2>
<h2 id="sequencing-platforme"><a aria-hidden="true" class="anchor-heading icon-link" href="#sequencing-platforme"></a>Sequencing platforme</h2>
<p>Les plateformes de sÃ©quenÃ§age Illumina HiSeq, PacBio Sequel II, et Nanopore PromethION sont des technologies avancÃ©es utilisÃ©es pour le sÃ©quenÃ§age de l'ADN, mais elles se diffÃ©rencient en termes de mÃ©thode de sÃ©quenÃ§age, de prÃ©cision, de longueur de lecture, et de coÃ»t. Voici un aperÃ§u de chaque plateforme :</p>
<ol>
<li>Illumina HiSeq Platform
Technologie de SÃ©quenÃ§age : Illumina utilise une mÃ©thode de "sÃ©quenÃ§age par synthÃ¨se". Les brins d'ADN sont fragmentÃ©s et chaque fragment est amplifiÃ© pour former des clusters. Lors du sÃ©quenÃ§age, des bases fluorescentes sont ajoutÃ©es une Ã  une, et chaque fluorescence est capturÃ©e par une camÃ©ra.
Longueur de Lecture : Cette plateforme produit gÃ©nÃ©ralement des lectures de courte longueur (entre 50 et 300 paires de bases par lecture).
Avantages : Haute prÃ©cision et faible taux d'erreur, adaptÃ©e aux projets nÃ©cessitant une couverture profonde, comme l'exome ou le gÃ©nome complet.
InconvÃ©nients : Longueur de lecture limitÃ©e, donc moins adaptÃ©e Ã  l'assemblage de gÃ©nomes complexes et Ã  la dÃ©tection de certaines variations structurelles.</li>
<li>PacBio Sequel II Platform
Technologie de SÃ©quenÃ§age : Utilise la technologie de sÃ©quenÃ§age par observation directe d'une enzyme polymÃ©rase, appelÃ©e SMRT (Single Molecule, Real-Time sequencing). L'ADN n'est pas amplifiÃ© ; la plateforme lit directement les longues molÃ©cules d'ADN.
Longueur de Lecture : Produits de trÃ¨s longues lectures (jusqu'Ã  100 000 bases), avec une longueur moyenne dâenviron 10 000-30 000 bases.
Avantages : AdaptÃ©e aux sÃ©quences complexes et aux longues rÃ©pÃ©titions, idÃ©ale pour lâassemblage de novo de gÃ©nomes et pour dÃ©tecter des variations structurelles.
InconvÃ©nients : CoÃ»t Ã©levÃ© par rapport aux plateformes Illumina, avec un taux dâerreur plus Ã©levÃ©, mÃªme s'il est rÃ©duit avec les lectures corrigÃ©es par consensus.</li>
<li>Nanopore PromethION Platform
Technologie de SÃ©quenÃ§age : La technologie de Nanopore permet de sÃ©quencer en temps rÃ©el en faisant passer une molÃ©cule d'ADN Ã  travers un nanopore et en mesurant les changements de courant Ã©lectrique pour identifier les bases.
Longueur de Lecture : Elle peut produire des lectures extrÃªmement longues, dÃ©passant plusieurs centaines de milliers de bases.
Avantages : Longueur de lecture quasiment illimitÃ©e et rapiditÃ© de sÃ©quenÃ§age en temps rÃ©el. Elle est portable et peut Ãªtre utilisÃ©e pour des sÃ©quenÃ§ages directement sur le terrain.
InconvÃ©nients : Taux d'erreur plus Ã©levÃ© que les autres plateformes, bien que les taux dâerreur aient diminuÃ© avec les amÃ©liorations technologiques.
En rÃ©sumÃ©, le choix de la plateforme dÃ©pendra du type de projet de sÃ©quenÃ§age : Illumina pour les projets nÃ©cessitant une haute prÃ©cision et une profondeur de couverture, PacBio pour des lectures longues et une bonne rÃ©solution des rÃ©gions rÃ©pÃ©tÃ©es, et Nanopore pour les lectures ultra-longues et les sÃ©quenÃ§ages rapides en temps rÃ©el.</li>
</ol>
<h2 id=""><a aria-hidden="true" class="anchor-heading icon-link" href="#"></a></h2>
<h2 id="lab-work-today"><a aria-hidden="true" class="anchor-heading icon-link" href="#lab-work-today"></a>Lab work today</h2>
<p>DNA extraction of CH2 and CH6, both liquid and solide culture
culture V1 from the stock to avoid contamination </p>
<h2 id="to-do-this-week"><a aria-hidden="true" class="anchor-heading icon-link" href="#to-do-this-week"></a>To do this week</h2>
<p>Storage all active strains in glycerol 70% at -80Â°C.</p>
<h2 id="claen-your-samples"><a aria-hidden="true" class="anchor-heading icon-link" href="#claen-your-samples"></a>claen your samples</h2>
<h2 id="today-30-11-2024"><a aria-hidden="true" class="anchor-heading icon-link" href="#today-30-11-2024"></a>Today 30-11-2024</h2>
<p>DNA extraction and purification of CH2, CH4 and V1
Results
PCR Does'n work for CH4 and V1 i have to redo extraction!</p>
<h2 id="tomorrow-31-10-24"><a aria-hidden="true" class="anchor-heading icon-link" href="#tomorrow-31-10-24"></a>Tomorrow 31-10-24</h2>
<p>storage of all strains
redo DNA isolation of CH4 solide and liquid media </p>
<h3 id="summary-of-the-week-10-26-2024"><a aria-hidden="true" class="anchor-heading icon-link" href="#summary-of-the-week-10-26-2024"></a>Summary of the Week (10-26-2024)</h3>
<p>I cultivated the remaining strains CH2, CH4, and V1, and performed a DNA extraction.
I successfully obtained the sequence for CH2 only. V1's DNA appeared fragmented on the agarose gel.
CH4 showed no detectable DNA, likely due to its very rigid cell wall, which may have hindered the extraction process.
Next week i will try other protocol for DNA extraction according to Floriane sugestions </p>
<h2 id="ch2-seq"><a aria-hidden="true" class="anchor-heading icon-link" href="#ch2-seq"></a>CH2 seq</h2>
<p>the seq was very clean means my culture is pure
CH2</p>
<blockquote>
<p>TCATTACAGAGTTGCAAAACTCCCCAAACCATCGTGAACGCTACCTAAACCGTTGCTTCGGCGGGCGGCGCCCTCGCGCGCCCCCCTGGGCCCGCACCGCGGGCGCCCGCCGGAGGTACACCAAACTCTTGATATGTCATGGCCACTCTGAGTCTTCTATACTGAATAAGTCAAAACTTTCAACAACGGATCTCTTGGTTCTGGCATCGATGAAGAACGCAGCGAAATGCGATAAGTAATGTGAATTGCAGAATTCAGTGAATCATCGAATCTTTGAACGCACATTGCGCCCGCCAGGATAGTATCCTGGCGGGCATGCCTGTTCGAGCGTCATTTCAACCCATCAAGCCCACGGCTTGTGTTGGGGACCTGCGGCTGCCCGCAGGCCCTGAAAACCAGTGGCGGGCTCGCTAGTCACACCGAGCGTAGTAGCATACGACCTCGCTCAGGGCGTGCTGCGGGTTCCAGCCGTAAAACGACCTTCGCGACCCCAAGGTTGACCTCGGATCAGGTAGGAGGACCCGCTGAACTTAAGCATATCAATAAGCGGAGGAA
<a title="Private" href="https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html" target="_blank" class="private">#NCBI (Private)</a> BLAST
<img src="/commons-dws-public/image-1.png" alt="alt text"></p>
</blockquote>
<p>Thermothelomyces guttulatus </p>
<p>WD query for compounds <a href="https://w.wiki/Bnes">https://w.wiki/Bnes</a></p>
<h2 id="monday"><a aria-hidden="true" class="anchor-heading icon-link" href="#monday"></a>Monday</h2>
<p>DNA extraction of CH4 and V1 </p>
<h2 id="active-strains-storage"><a aria-hidden="true" class="anchor-heading icon-link" href="#active-strains-storage"></a>active strains storage</h2>
<p>I stored all the active strains in 70% glycerol, each strain has 4 copies, and the tudes are at -80Â°C.
<img src="/commons-dws-public/conserved%20strains.jpg" alt="alt text"></p>
<h2 id="todays-review-04-11-2024"><a aria-hidden="true" class="anchor-heading icon-link" href="#todays-review-04-11-2024"></a>today's review (04-11-2024)</h2>
<p>Finaly i get bands for V1 and CH4 isolates
send for sequencing
all strains stored at -80Â°C.</p>
<h2 id="tommorow-02-11-2024"><a aria-hidden="true" class="anchor-heading icon-link" href="#tommorow-02-11-2024"></a>Tommorow 02-11-2024</h2>
<p>write on my lab note book
wait for results sequences.
clean cutures wastes
analyses ITS seq of CH4 and 16S for V1. </p>
<h2 id="we-have-sent-the-sequences-of-v1-and-ch4-to-sequencing-the-results-are-not-reliable-to-be-reextract"><a aria-hidden="true" class="anchor-heading icon-link" href="#we-have-sent-the-sequences-of-v1-and-ch4-to-sequencing-the-results-are-not-reliable-to-be-reextract"></a>We have sent the sequences of V1 and CH4 to sequencing, the results are not reliable, to be reextract.</h2>
<p>results recieved 05-11-2024
CH4
CATTACCGAKTGCGGGCTGCCTCCGGGCGCCCAACCTCCCWCCCGTGACTACCTAACACTGTTGCTTCRGCGGGGAGCCCCCCAGGGGGCGAGCCGCCGGGGACCACTGAACTTCATGCCTGASAGKGATGCAGWCTGASCCTGAATACAAATCAGTCAAAACTTTCAACAATGGATCTCTTGGTTCCGGCATCRATGAAGAACGCAGCGAACTGC</p>
<p>L'intensitÃ© du signal de votre rÃ©action de sÃ©quenÃ§age est trÃ¨s faible. Par consÃ©quent, le rÃ©sultat de votre sÃ©quenÃ§age doit Ãªtre soigneusement Ã©valuÃ©.
RÃ©sultat de sÃ©quenÃ§age modÃ©rÃ© ou ambigu. Veuillez vÃ©rifier l'Ã©tat de votre rÃ©action, qui sera mise Ã  jour dans la zone de tÃ©lÃ©chargement dans le courant de la matinÃ©e (jours ouvrables uniquement).
Economy Run (Tube)
13787654 94799985
V1
NNNNN</p>
<h2 id="today-06-11-2024"><a aria-hidden="true" class="anchor-heading icon-link" href="#today-06-11-2024"></a>Today 06-11-2024</h2>
<p>-DNA extraction of CH4 AND V1</p>
<h2 id="tomorrow-07-11-2024"><a aria-hidden="true" class="anchor-heading icon-link" href="#tomorrow-07-11-2024"></a>Tomorrow 07-11-2024</h2>
<p>PCR, agarose gel and purification.</p>
<h2 id="revievs-07-11-2024"><a aria-hidden="true" class="anchor-heading icon-link" href="#revievs-07-11-2024"></a>Reviev's 07-11-2024</h2>
<hr>
<strong>Children</strong>
<ol>
<li><a href="/commons-dws-public/notes/kvtssreflme8xid39zy3t0n">Streptomyces lusitanus</a></li>
</ol></div></div><div style="padding-left:10px;padding-right:10px" class="ant-col ant-col-xs-0 ant-col-md-6"><div><div class=""><div class="ant-anchor-wrapper dendron-toc" style="max-height:calc(100vh - 64px);z-index:1"><div class="ant-anchor"><div class="ant-anchor-ink"><span class="ant-anchor-ink-ball"></span></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#streptomyces-lusitanus" title="Streptomyces lusitanus">Streptomyces lusitanus</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#streptomyces-xanthophaeus" title="Streptomyces xanthophaeus">Streptomyces xanthophaeus</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#paecilomyces-lilacinus" title="Paecilomyces lilacinus">Paecilomyces lilacinus</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#16s-dna-sequences" title="16S DNA sequences">16S DNA sequences</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#its-fungi-sequences" title="ITS Fungi sequences">ITS Fungi sequences</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#dna-protocol-extraction" title="DNA Protocol extraction">DNA Protocol extraction</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#strains-picture" title="Strains picture">Strains picture</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#to-do-next-week" title="To do next week">To do next week</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#saturday" title="Saturday">Saturday</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#fungal-dna-extraction-protocol-without-kit" title="Fungal DNA extraction protocol without kit">Fungal DNA extraction protocol without kit</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#culture-v1-in-tsb-for-tomorrow" title="Culture V1 in TSB for tomorrow">Culture V1 in TSB for tomorrow</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#sequencing-platforme" title="Sequencing platforme">Sequencing platforme</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#" title=""></a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#lab-work-today" title="Lab work today">Lab work today</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#to-do-this-week" title="To do this week">To do this week</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#claen-your-samples" title="claen your samples">claen your samples</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#today-30-11-2024" title="Today 30-11-2024">Today 30-11-2024</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#tomorrow-31-10-24" title="Tomorrow 31-10-24">Tomorrow 31-10-24</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#summary-of-the-week-10-26-2024" title="Summary of the Week (10-26-2024)">Summary of the Week (10-26-2024)</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#ch2-seq" title="CH2 seq">CH2 seq</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#monday" title="Monday">Monday</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#active-strains-storage" title="active strains storage">active strains storage</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#todays-review-04-11-2024" title="today&#x27;s review (04-11-2024)">today&#x27;s review (04-11-2024)</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#tommorow-02-11-2024" title="Tommorow 02-11-2024">Tommorow 02-11-2024</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#we-have-sent-the-sequences-of-v1-and-ch4-to-sequencing-the-results-are-not-reliable-to-be-reextract" title="We have sent the sequences of V1 and CH4 to sequencing, the results are not reliable, to be reextract.">We have sent the sequences of V1 and CH4 to sequencing, the results are not reliable, to be reextract.</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#today-06-11-2024" title="Today 06-11-2024">Today 06-11-2024</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#tomorrow-07-11-2024" title="Tomorrow 07-11-2024">Tomorrow 07-11-2024</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#revievs-07-11-2024" title="Reviev&#x27;s 07-11-2024">Reviev&#x27;s 07-11-2024</a></div></div></div></div></div></div></div></div></div></div></div><div class="ant-divider ant-divider-horizontal" role="separator"></div><footer class="ant-layout-footer" style="padding:0 24px 24px"></footer></main></section></section></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{"note":{"id":"a0iu9pk5ktxoo6h7y7gmm6e","title":"Strain","desc":"","updated":1731936637171,"created":1729759559391,"custom":{},"fname":"projects.commons-public.microcosmics.strains","type":"note","vault":{"fsPath":"vault"},"contentHash":"e654caf53a11ae11d48131f7eea990cb","links":[{"type":"wiki","from":{"fname":"projects.commons-public.microcosmics.strains","id":"a0iu9pk5ktxoo6h7y7gmm6e","vaultName":"vault"},"value":"tags.wd","alias":"#wd","position":{"start":{"line":12,"column":1,"offset":198},"end":{"line":12,"column":4,"offset":201},"indent":[]},"xvault":false,"to":{"fname":"tags.wd"}},{"type":"wiki","from":{"fname":"projects.commons-public.microcosmics.strains","id":"a0iu9pk5ktxoo6h7y7gmm6e","vaultName":"vault"},"value":"projects.commons-public.microcosmics.strains.Streptomyces lusitanus","position":{"start":{"line":14,"column":31,"offset":234},"end":{"line":14,"column":102,"offset":305},"indent":[]},"xvault":false,"sameFile":false,"to":{"fname":"projects.commons-public.microcosmics.strains.Streptomyces lusitanus"}},{"type":"wiki","from":{"fname":"projects.commons-public.microcosmics.strains","id":"a0iu9pk5ktxoo6h7y7gmm6e","vaultName":"vault"},"value":"tags.NCBI","alias":"#NCBI","position":{"start":{"line":178,"column":1,"offset":13051},"end":{"line":178,"column":6,"offset":13056},"indent":[]},"xvault":false,"to":{"fname":"tags.NCBI"}}],"anchors":{"streptomyces-lusitanus":{"type":"header","text":"Streptomyces lusitanus","value":"streptomyces-lusitanus","line":11,"column":0,"depth":2},"streptomyces-xanthophaeus":{"type":"header","text":"Streptomyces xanthophaeus","value":"streptomyces-xanthophaeus","line":22,"column":0,"depth":2},"paecilomyces-lilacinus":{"type":"header","text":"Paecilomyces lilacinus","value":"paecilomyces-lilacinus","line":29,"column":0,"depth":2},"16s-dna-sequences":{"type":"header","text":"16S DNA sequences","value":"16s-dna-sequences","line":35,"column":0,"depth":2},"its-fungi-sequences":{"type":"header","text":"ITS Fungi sequences","value":"its-fungi-sequences","line":40,"column":0,"depth":2},"dna-protocol-extraction":{"type":"header","text":"DNA Protocol extraction","value":"dna-protocol-extraction","line":45,"column":0,"depth":2},"strains-picture":{"type":"header","text":"Strains picture","value":"strains-picture","line":96,"column":0,"depth":2},"to-do-next-week":{"type":"header","text":"To do next week","value":"to-do-next-week","line":117,"column":0,"depth":2},"saturday":{"type":"header","text":"Saturday","value":"saturday","line":121,"column":0,"depth":2},"fungal-dna-extraction-protocol-without-kit":{"type":"header","text":"Fungal DNA extraction protocol without kit","value":"fungal-dna-extraction-protocol-without-kit","line":123,"column":0,"depth":2},"culture-v1-in-tsb-for-tomorrow":{"type":"header","text":"Culture V1 in TSB for tomorrow","value":"culture-v1-in-tsb-for-tomorrow","line":138,"column":0,"depth":2},"sequencing-platforme":{"type":"header","text":"Sequencing platforme","value":"sequencing-platforme","line":140,"column":0,"depth":2},"":{"type":"header","text":"","value":"","line":158,"column":0,"depth":2},"lab-work-today":{"type":"header","text":"Lab work today","value":"lab-work-today","line":160,"column":0,"depth":2},"to-do-this-week":{"type":"header","text":"To do this week","value":"to-do-this-week","line":163,"column":0,"depth":2},"claen-your-samples":{"type":"header","text":"claen your samples","value":"claen-your-samples","line":165,"column":0,"depth":2},"today-30-11-2024":{"type":"header","text":"Today 30-11-2024","value":"today-30-11-2024","line":166,"column":0,"depth":2},"tomorrow-31-10-24":{"type":"header","text":"Tomorrow 31-10-24","value":"tomorrow-31-10-24","line":170,"column":0,"depth":2},"summary-of-the-week-10-26-2024":{"type":"header","text":"Summary of the Week (10-26-2024)","value":"summary-of-the-week-10-26-2024","line":173,"column":0,"depth":3},"ch2-seq":{"type":"header","text":"CH2 seq","value":"ch2-seq","line":180,"column":0,"depth":2},"monday":{"type":"header","text":"Monday","value":"monday","line":191,"column":0,"depth":2},"active-strains-storage":{"type":"header","text":"active strains storage","value":"active-strains-storage","line":193,"column":0,"depth":2},"todays-review-04-11-2024":{"type":"header","text":"today's review (04-11-2024)","value":"todays-review-04-11-2024","line":197,"column":0,"depth":2},"tommorow-02-11-2024":{"type":"header","text":"Tommorow 02-11-2024","value":"tommorow-02-11-2024","line":201,"column":0,"depth":2},"we-have-sent-the-sequences-of-v1-and-ch4-to-sequencing-the-results-are-not-reliable-to-be-reextract":{"type":"header","text":"We have sent the sequences of V1 and CH4 to sequencing, the results are not reliable, to be reextract.","value":"we-have-sent-the-sequences-of-v1-and-ch4-to-sequencing-the-results-are-not-reliable-to-be-reextract","line":207,"column":0,"depth":2},"today-06-11-2024":{"type":"header","text":"Today 06-11-2024","value":"today-06-11-2024","line":219,"column":0,"depth":2},"tomorrow-07-11-2024":{"type":"header","text":"Tomorrow 07-11-2024","value":"tomorrow-07-11-2024","line":221,"column":0,"depth":2},"revievs-07-11-2024":{"type":"header","text":"Reviev's 07-11-2024","value":"revievs-07-11-2024","line":223,"column":0,"depth":2}},"children":["kvtssreflme8xid39zy3t0n"],"parent":"0zmqus0478ehzat7p16a7sr","data":{}},"body":"\u003ch1 id=\"strain\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#strain\"\u003e\u003c/a\u003eStrain\u003c/h1\u003e\n\u003cp\u003eThe followoing strains have been identified through their 16RNA sequencing data.\u003c/p\u003e\n\u003ch2 id=\"streptomyces-lusitanus\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#streptomyces-lusitanus\"\u003e\u003c/a\u003eStreptomyces lusitanus\u003c/h2\u003e\n\u003cp\u003eWD query for compounds \u003ca href=\"https://w.wiki/BfHS\"\u003ehttps://w.wiki/BfHS\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e30 compound isolated \u003ca href=\"https://w.wiki/BfHK\"\u003ehttps://w.wiki/BfHK\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e\u003ca title=\"Private\" href=\"https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html\" target=\"_blank\" class=\"private\"\u003e#wd (Private)\u003c/a\u003e \u003c/p\u003e\n\u003cp\u003eMore info in the related note \u003ca href=\"/commons-dws-public/notes/kvtssreflme8xid39zy3t0n\"\u003eStreptomyces lusitanus\u003c/a\u003e\u003c/p\u003e\n\u003ch2 id=\"streptomyces-xanthophaeus\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#streptomyces-xanthophaeus\"\u003e\u003c/a\u003eStreptomyces xanthophaeus\u003c/h2\u003e\n\u003cp\u003eWD query for compounds \u003ca href=\"https://w.wiki/BfHN\"\u003ehttps://w.wiki/BfHN\u003c/a\u003e\n\u003ca href=\"https://scholia.toolforge.org/taxon/Q26293525#metabolome\"\u003ehttps://scholia.toolforge.org/taxon/Q26293525#metabolome\u003c/a\u003e\u003c/p\u003e\n\u003ch2 id=\"paecilomyces-lilacinus\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#paecilomyces-lilacinus\"\u003e\u003c/a\u003ePaecilomyces lilacinus\u003c/h2\u003e\n\u003cp\u003eWD query for compounds \u003ca href=\"https://w.wiki/BfHT\"\u003ehttps://w.wiki/BfHT\u003c/a\u003e\nNot to be worked on.\nHuman pathogen.\u003c/p\u003e\n\u003ch2 id=\"16s-dna-sequences\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#16s-dna-sequences\"\u003e\u003c/a\u003e16S DNA sequences\u003c/h2\u003e\n\u003cblockquote\u003e\n\u003cp\u003eS3\nAAGTCGAACGATGAACCACTTCGGTGGGGATTAGTGGCGAACGGGTGAGTAACACGTGGGCAATCTGCCCTGCACTCTGGGACAAGCCCTGGAAACGGGGTCTAATACCGGATACTGATCGCCTTGGGCATCCTTGGTGATCGAAAGCTCCGGCGGTGCAGGATGAGCCCGCGGCCTATCAGCTTGTTGGTGAGGTAATGGCTCACCAAGGCGACGACGGGTAGCCGGCCTGAGAGGGCGACCGGCCACACTGGGACTGAGACACGGCCCAGACTCCTACGGGAGGCAGCAGTGGGGAATATTGCACAATGGGCGAAAGCCTGATGCAGCGACGCCGCGTGAGGGATGACGGCCTTCGGGTTGTAAACCTCTTTCAGCAGGGAAGAAGCGAAAGTGACGGTACCTGCAGAAGAAGCGCCGGCTAACTACGTGCCAGCAGCCGCGGTAATACGTAGGGCGCGAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGCTTGTCGCGTCGGTTGTGAAAGCCCGGGGCTTAACCCCGGGTCTGCAGTCGATACGGGCAGGCTAGAGTTCGGTAGGGGAGATCGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGATCTCTGGGCCGATACTGACGCTGAGGAGCGAAAGCGTGGGGAGCGAACAGGATTAGATACCCTGGTAGTCCACGCCGTAAACGGTGGGCACTAGGTGTGGGCGACATTCCACGTCGTCCGTGCCGCAGCTAACGCATTAAGTGCCCCGCCTGGGGAGTACGGCCGCAAGGCTAAAACTCAAAGGAATTGACGGGGGCCCGCACAAGCGGCGGAGCATGTGGCTTAATTCGACGCAACGCGAAGAACCTTACCAAGGCTTGACATACACCGGAAACGCCCGGAGATGGGCGCCCCTTGTGGTCGGAGTACAGGTGGTGCATGGCTGTCGTCAGCTCGTGTCGTGAGATGTTGGGTTAAGTCCCGCAACGAGCGCACCCTTGTCCCGTGTTGCCAGCAGGCCCTTGGGGTGCTGGGAATCACGGGAGACCGCCGGGGTCAATCGGAGGAAGGGGGG\nS6\nATGCAGTCGAACGATGAAGCCCTTCGGGGTGGATTAGTGGCGAACGGGTGAGTAACACGTGGGCAATCTGCCCTTCACTCTGGGACAAGCCCTGGAAACGGGGTCTAATACCGGATACCACTCCTGCCTGCATGGGCGGGGGTTGAAAGCTCCGGCGGTGAAGGATGAGCCCGCGGCCTATCAGCTTGTTGGTGGGGTAATGGCCCACCAAGGCGACGACGGGTAGCCGGCCTGAGAGGGCGACCGGCCACACTGGGACTGAGACACGGCCCAGACTCCTACGGGAGGCAGCAGTGGGGAATATTGCACAATGGGCGAAAGCCTGATGCAGCGACGCCGCGTGAGGGATGACGGCCTTCGGGTTGTAAACCTCTTTCAGCAGGGAAGAAGCGAAAGTGACGGTACCTGCAGAAGAAGCGCCGGCTAACTACGTGCCAGCAGCCGCGGTAATACGTAGGGCGCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGCTTGTCACGTCGGATGTGAAAGCCCGAGGCTTAACCTCGGGTCTGCATTCGATACGGGCTAGCTAGAGTGTGGTAGGGGAGATCGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGATCTCTGGGCCATTACTGACGCTGAGGAGCGAAAGCGTGGGGAGCGAACAGGATTAGATACCCTGGTAGTCCACGCCGTAAACGTTGGGAACTAGGTGTTGGCGACATTCCACGTCGTCGGTGCCGCAGCTAACGCATTAAGTTCCCCGCCTGGGGAGTACGGCCGCAAGGCTAAAACTCAAAGGAATTGACGGGGGCCCGCACAAGCGGCGGAGCATGTGGCTTAATTCGACGCAACGCGAAGAACCTTACCAAGGCTTGACATATACCGGAAAGCATTAGAGATAGTGCCCCCCTTGTGGTCGGTATACAGGTGGTGCATGGCTGTCGTCAGCTCGTGTCGTGAGATGTTGGGTTAAGTCCCGCAACGAGCGCAACCCTTGTCCTGTGTTGCCAGCATGCCCTTCGGGGTGATGGGGACTCACAGGAGACCGCCGGGGTCAACTCGGAGGAGGTGGGGACGACGTCAGTCATCATGCCCCTTAG\u003c/p\u003e\n\u003ch2 id=\"its-fungi-sequences\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#its-fungi-sequences\"\u003e\u003c/a\u003eITS Fungi sequences\u003c/h2\u003e\n\u003cp\u003eCH6\nAGCGGAGGGATCATTACCGAGTTATACAACTCCCAAACCCACTGTGAACCTTACCTCAGTTGCCTCGGCGGGAACGCCCCGGCCGCCTGCCCCCGCGCCGGCGCCGGACCCAGGCGCCCGCCGCAGGGACCCCAAACTCTCTTGCATTACGCCCAGCGGGCGGAATTTCTTCTCTGAGTTGCACAAGCAAAAACAAATGAATCAAAACTTTCAACAACGGATCTCTTGGTTCTGGCATCGATGAAGAACGCAGCGAAATGCGATAAGTAATGTGAATTGCAGAATTCAGTGAATCATCGAATCTTTGAACGCACATTGCGCCCGCCAGCATTCTGGCGGGCATGCCTGTTCGAGCGTCATTTCAACCCTCGAGCCCCCCCCGGGGGCCTCGGTGTTGGGGGACGGCACACCAGCCGCCCCCGAAATGCAGTGGCGACCCCGCCGCAGCCTCCCCTGCGTAGTAGCACACACCTCGCACCGGAGCGCGGAGGCGGTCACGCCGTAAAACGCCCAACTTTCTTAGAGTTGACCTCGGATCAGGTAGGAATACCCGCTGAACTTAAGCATATCAATAAGCGGAGGAA\u003c/p\u003e\n\u003c/blockquote\u003e\n\u003ch2 id=\"dna-protocol-extraction\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#dna-protocol-extraction\"\u003e\u003c/a\u003eDNA Protocol extraction\u003c/h2\u003e\n\u003cp\u003ePCR PROTOCOL FOR Actinomycetes\nDNA extraction (heat shock method):\u003c/p\u003e\n\u003col\u003e\n\u003cli\u003eTake bacterial mycelium in a clean 1.5 mL Eppendorf tube.\u003c/li\u003e\n\u003cli\u003eGrind the cells using a clean tip and suspend them thoroughly in 1 mL of distilled water. \u003c/li\u003e\n\u003cli\u003eHeat the samples at 95Â°C in the heating block for 20 min.\u003c/li\u003e\n\u003cli\u003eTransfer the tube (suddenly) to ice for 10 min. \u003c/li\u003e\n\u003cli\u003eRe-heat the sample in the heating block at 95Â°C for 10 min.\u003c/li\u003e\n\u003cli\u003eStore DNA at -20Â°C.\u003c/li\u003e\n\u003c/ol\u003e\n\u003cp\u003e16S rDNA Primers:\nâ¢\t27F and 1492R\n27F: 5â-AGAGTTTGATCCTGGCTCAG-3â\n1492R: 5â-CGGTTACCTTGTTACGACTT-3â\n(A)\t PCR amplification in 25 Î¼l of reaction (Accustart II):\nâ¢\t5 Î¼L of genomic DNA\nâ¢\t0.5 Î¼L of primer F (25 Î¼M)\nâ¢\t0.5 Î¼L of primer R (25 Î¼M)\nâ¢\t12.5 Î¼L of Accustart II PCR toughmix (2x)\nâ¢\t0.5 Î¼L of gel track loading Dye (50X)\nâ¢\t6 Î¼L of water\u003c/p\u003e\n\u003cp\u003e(B)\tPCR amplification in 25 Î¼l of reaction (Solifast):\nâ¢\t5 Î¼L of genomic DNA\nâ¢\t0.5 Î¼L of primer F (25 Î¼M)\nâ¢\t0.5 Î¼L of primer R (25 Î¼M)\nâ¢\t5 Î¼L of Solifast (5x)\nâ¢\t14 Î¼L of water\u003c/p\u003e\n\u003cp\u003eAmplification conditions:\nâ¢\tInitial denaturation at 94Â°C for 3 min\nâ¢\tFollowed by 35 cycles of\u003c/p\u003e\n\u003col\u003e\n\u003cli\u003ea denaturation step at 94Â°C for 30 s,\u003c/li\u003e\n\u003cli\u003ean annealing step at 56Â°C for 30 s, and\u003c/li\u003e\n\u003cli\u003ean extension step at 72Â°C for 1 min 40 s,\nâ¢\tFollowed by a final extension step at 72Â°C for 10 min.\u003c/li\u003e\n\u003c/ol\u003e\n\u003cp\u003eGel electrophoresis:\u003c/p\u003e\n\u003col\u003e\n\u003cli\u003ePrepare 1% agarose: 1g agarose in 100 mL of 1X TAE buffer (40 mol/L Tris-acetate, and 1 mM EDTA, pH 8.0). (0.6 g in 60 mL for small trays)\u003c/li\u003e\n\u003cli\u003eStain the gel with 6 Î¼l of GreenSafe solution (3 for 60 mL).\u003c/li\u003e\n\u003cli\u003eImmerse the gel in 1X TAE (pH 8.0) buffer.\u003c/li\u003e\n\u003cli\u003eLoad the prepared PCR products into the wells. \u003c/li\u003e\n\u003cli\u003e Use a 1 Kbp-DNA ladder as a molecular weight marker. \u003c/li\u003e\n\u003cli\u003ePass 100 volts of electric current through the gel for 30 min.\u003c/li\u003e\n\u003cli\u003eRemove and visualize the gel, in a Trans illumination cabinet.\u003c/li\u003e\n\u003cli\u003eCapture the image using a gel documentation system. \u003c/li\u003e\n\u003c/ol\u003e\n\u003cp\u003eâ¢\tAppearance of the target band specified for the primer set on the agarose gel is considered as a positive amplification product.\u003c/p\u003e\n\u003ch2 id=\"strains-picture\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#strains-picture\"\u003e\u003c/a\u003eStrains picture\u003c/h2\u003e\n\u003cp\u003eCH6\n\u003cimg src=\"/commons-dws-public/photo_57@22-10-2024_09-06-59_thumb.jpg\" alt=\"alt text\"\u003e\n\u003cimg src=\"/commons-dws-public/photo_92@22-10-2024_09-13-16_thumb.jpg\" alt=\"alt text\"\u003e\n\u003cimg src=\"/commons-dws-public/photo_71@22-10-2024_09-07-00_thumb.jpg\" alt=\"alt text\"\u003e\u003c/p\u003e\n\u003cp\u003eS3\u003c/p\u003e\n\u003cp\u003e\u003cimg src=\"/commons-dws-public/photo_97@22-10-2024_09-13-17_thumb.jpg\" alt=\"alt text\"\u003e\u003c/p\u003e\n\u003cp\u003eS6\n\u003cimg src=\"/commons-dws-public/photo_53@22-10-2024_09-06-59.jpg\" alt=\"alt text\"\u003e\u003c/p\u003e\n\u003cp\u003eCH4\n\u003cimg src=\"/commons-dws-public/photo_86@22-10-2024_09-13-09_thumb.jpg\" alt=\"alt text\"\u003e\n\u003cimg src=\"/commons-dws-public/photo_63@22-10-2024_09-07-00_thumb.jpg\" alt=\"alt text\"\u003e\n\u003cimg src=\"/commons-dws-public/photo_108@22-10-2024_09-13-17.jpg\" alt=\"alt text\"\u003e\nV1\n\u003cimg src=\"/commons-dws-public/photo_59@22-10-2024_09-06-59_thumb.jpg\" alt=\"alt text\"\u003e\u003c/p\u003e\n\u003ch2 id=\"to-do-next-week\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#to-do-next-week\"\u003e\u003c/a\u003eTo do next week\u003c/h2\u003e\n\u003cp\u003e1- Saturday culture bacteria\n1- run LC-MS for vials soil\n2- ITS extraction of CH4, CH2, V1\u003c/p\u003e\n\u003ch2 id=\"saturday\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#saturday\"\u003e\u003c/a\u003eSaturday\u003c/h2\u003e\n\u003cp\u003eCulture CH2 and CH4 \u003c/p\u003e\n\u003ch2 id=\"fungal-dna-extraction-protocol-without-kit\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#fungal-dna-extraction-protocol-without-kit\"\u003e\u003c/a\u003eFungal DNA extraction protocol without kit\u003c/h2\u003e\n\u003cul\u003e\n\u003cli\u003eProtocol 1\nAll fungal isolates were treated with 70% ethanol and heat inactivated (100Â°C for 30 min). A bead beating step was added to assist with mechanical lysis of the fungal cell wall. A viability study showed that these steps resulted in no growth of a variety of diverse clinically relevant fungi, including both yeasts and molds (Supplemental Table S2). For all samples, the Qiagen (Valencia, CA, USA) EZ1 Blood and Tissue Kit as well as the EZ1 Advanced XL instrument were used (following manufacturerâs instructions) to extract genomic DNA from fungal isolates. Post extraction, DNA was quantified using Qubit 1Ã double-stranded DNA HS assay with the Qubit 3.0 Fluorometer (Thermo Fisher Scientific, Waltham, MA, USA). DNA quantities greater than 0.04 ng/ÂµL were considered acceptable. PCR grade water was used as a negative extraction and sequencing control. A reference strain of Candida tropicalis (CDC AR Bank 3045) was used as a sequencing positive control for each run [ doi: 10.3390/jof9020183].\nJe rÃ©sume:\u003c/li\u003e\n\u003c/ul\u003e\n\u003col\u003e\n\u003cli\u003e\n\u003cp\u003eInactivation des isolats : Tous les isolats fongiques ont Ã©tÃ© traitÃ©s avec de l'Ã©thanol Ã  70 % et chauffÃ©s Ã  100Â°C pendant 30 minutes pour Ãªtre inactivÃ©s. Ce traitement par la chaleur et l'alcool Ã©limine la viabilitÃ© des champignons, empÃªchant leur croissance, ce qui est confirmÃ© par une Ã©tude montrant l'absence de croissance aprÃ¨s ces Ã©tapes, mÃªme pour des champignons cliniquement importants (levures et moisissures).\u003c/p\u003e\n\u003c/li\u003e\n\u003cli\u003e\n\u003cp\u003eLyse mÃ©canique : Une Ã©tape de broyage avec des billes (\"bead beating\") est ajoutÃ©e pour aider Ã  briser la paroi cellulaire des champignons, qui est trÃ¨s rÃ©sistante. Cette Ã©tape facilite la libÃ©ration de l'ADN contenu Ã  l'intÃ©rieur des cellules fongiques.\u003c/p\u003e\n\u003c/li\u003e\n\u003cli\u003e\n\u003cp\u003eExtraction de l'ADN : Le kit Qiagen EZ1 Blood and Tissue et lâinstrument EZ1 Advanced XL ont Ã©tÃ© utilisÃ©s pour extraire l'ADN gÃ©nomique des isolats, en suivant les instructions du fabricant.\u003c/p\u003e\n\u003c/li\u003e\n\u003cli\u003e\n\u003cp\u003eQuantification de l'ADN : AprÃ¨s l'extraction, la quantitÃ© d'ADN est mesurÃ©e avec le test Qubit 1Ã double-brin dâADN HS et le fluoromÃ¨tre Qubit 3.0. Seules les quantitÃ©s supÃ©rieures Ã  0,04 ng/ÂµL sont considÃ©rÃ©es comme acceptables pour les analyses suivantes.\u003c/p\u003e\n\u003c/li\u003e\n\u003c/ol\u003e\n\u003cp\u003e5.ContrÃ´les de qualitÃ© : De l'eau de qualitÃ© PCR est utilisÃ©e comme contrÃ´le nÃ©gatif pour s'assurer qu'aucune contamination n'est introduite pendant l'extraction et le sÃ©quenÃ§age. Une souche de rÃ©fÃ©rence de Candida tropicalis (CDC AR Bank 3045) sert de contrÃ´le positif pour vÃ©rifier la qualitÃ© et la prÃ©cision de chaque sÃ©quenÃ§age.\u003c/p\u003e\n\u003ch2 id=\"culture-v1-in-tsb-for-tomorrow\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#culture-v1-in-tsb-for-tomorrow\"\u003e\u003c/a\u003eCulture V1 in TSB for tomorrow\u003c/h2\u003e\n\u003ch2 id=\"sequencing-platforme\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#sequencing-platforme\"\u003e\u003c/a\u003eSequencing platforme\u003c/h2\u003e\n\u003cp\u003eLes plateformes de sÃ©quenÃ§age Illumina HiSeq, PacBio Sequel II, et Nanopore PromethION sont des technologies avancÃ©es utilisÃ©es pour le sÃ©quenÃ§age de l'ADN, mais elles se diffÃ©rencient en termes de mÃ©thode de sÃ©quenÃ§age, de prÃ©cision, de longueur de lecture, et de coÃ»t. Voici un aperÃ§u de chaque plateforme :\u003c/p\u003e\n\u003col\u003e\n\u003cli\u003eIllumina HiSeq Platform\nTechnologie de SÃ©quenÃ§age : Illumina utilise une mÃ©thode de \"sÃ©quenÃ§age par synthÃ¨se\". Les brins d'ADN sont fragmentÃ©s et chaque fragment est amplifiÃ© pour former des clusters. Lors du sÃ©quenÃ§age, des bases fluorescentes sont ajoutÃ©es une Ã  une, et chaque fluorescence est capturÃ©e par une camÃ©ra.\nLongueur de Lecture : Cette plateforme produit gÃ©nÃ©ralement des lectures de courte longueur (entre 50 et 300 paires de bases par lecture).\nAvantages : Haute prÃ©cision et faible taux d'erreur, adaptÃ©e aux projets nÃ©cessitant une couverture profonde, comme l'exome ou le gÃ©nome complet.\nInconvÃ©nients : Longueur de lecture limitÃ©e, donc moins adaptÃ©e Ã  l'assemblage de gÃ©nomes complexes et Ã  la dÃ©tection de certaines variations structurelles.\u003c/li\u003e\n\u003cli\u003ePacBio Sequel II Platform\nTechnologie de SÃ©quenÃ§age : Utilise la technologie de sÃ©quenÃ§age par observation directe d'une enzyme polymÃ©rase, appelÃ©e SMRT (Single Molecule, Real-Time sequencing). L'ADN n'est pas amplifiÃ© ; la plateforme lit directement les longues molÃ©cules d'ADN.\nLongueur de Lecture : Produits de trÃ¨s longues lectures (jusqu'Ã  100 000 bases), avec une longueur moyenne dâenviron 10 000-30 000 bases.\nAvantages : AdaptÃ©e aux sÃ©quences complexes et aux longues rÃ©pÃ©titions, idÃ©ale pour lâassemblage de novo de gÃ©nomes et pour dÃ©tecter des variations structurelles.\nInconvÃ©nients : CoÃ»t Ã©levÃ© par rapport aux plateformes Illumina, avec un taux dâerreur plus Ã©levÃ©, mÃªme s'il est rÃ©duit avec les lectures corrigÃ©es par consensus.\u003c/li\u003e\n\u003cli\u003eNanopore PromethION Platform\nTechnologie de SÃ©quenÃ§age : La technologie de Nanopore permet de sÃ©quencer en temps rÃ©el en faisant passer une molÃ©cule d'ADN Ã  travers un nanopore et en mesurant les changements de courant Ã©lectrique pour identifier les bases.\nLongueur de Lecture : Elle peut produire des lectures extrÃªmement longues, dÃ©passant plusieurs centaines de milliers de bases.\nAvantages : Longueur de lecture quasiment illimitÃ©e et rapiditÃ© de sÃ©quenÃ§age en temps rÃ©el. Elle est portable et peut Ãªtre utilisÃ©e pour des sÃ©quenÃ§ages directement sur le terrain.\nInconvÃ©nients : Taux d'erreur plus Ã©levÃ© que les autres plateformes, bien que les taux dâerreur aient diminuÃ© avec les amÃ©liorations technologiques.\nEn rÃ©sumÃ©, le choix de la plateforme dÃ©pendra du type de projet de sÃ©quenÃ§age : Illumina pour les projets nÃ©cessitant une haute prÃ©cision et une profondeur de couverture, PacBio pour des lectures longues et une bonne rÃ©solution des rÃ©gions rÃ©pÃ©tÃ©es, et Nanopore pour les lectures ultra-longues et les sÃ©quenÃ§ages rapides en temps rÃ©el.\u003c/li\u003e\n\u003c/ol\u003e\n\u003ch2 id=\"\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#\"\u003e\u003c/a\u003e\u003c/h2\u003e\n\u003ch2 id=\"lab-work-today\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#lab-work-today\"\u003e\u003c/a\u003eLab work today\u003c/h2\u003e\n\u003cp\u003eDNA extraction of CH2 and CH6, both liquid and solide culture\nculture V1 from the stock to avoid contamination \u003c/p\u003e\n\u003ch2 id=\"to-do-this-week\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#to-do-this-week\"\u003e\u003c/a\u003eTo do this week\u003c/h2\u003e\n\u003cp\u003eStorage all active strains in glycerol 70% at -80Â°C.\u003c/p\u003e\n\u003ch2 id=\"claen-your-samples\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#claen-your-samples\"\u003e\u003c/a\u003eclaen your samples\u003c/h2\u003e\n\u003ch2 id=\"today-30-11-2024\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#today-30-11-2024\"\u003e\u003c/a\u003eToday 30-11-2024\u003c/h2\u003e\n\u003cp\u003eDNA extraction and purification of CH2, CH4 and V1\nResults\nPCR Does'n work for CH4 and V1 i have to redo extraction!\u003c/p\u003e\n\u003ch2 id=\"tomorrow-31-10-24\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#tomorrow-31-10-24\"\u003e\u003c/a\u003eTomorrow 31-10-24\u003c/h2\u003e\n\u003cp\u003estorage of all strains\nredo DNA isolation of CH4 solide and liquid media \u003c/p\u003e\n\u003ch3 id=\"summary-of-the-week-10-26-2024\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#summary-of-the-week-10-26-2024\"\u003e\u003c/a\u003eSummary of the Week (10-26-2024)\u003c/h3\u003e\n\u003cp\u003eI cultivated the remaining strains CH2, CH4, and V1, and performed a DNA extraction.\nI successfully obtained the sequence for CH2 only. V1's DNA appeared fragmented on the agarose gel.\nCH4 showed no detectable DNA, likely due to its very rigid cell wall, which may have hindered the extraction process.\nNext week i will try other protocol for DNA extraction according to Floriane sugestions \u003c/p\u003e\n\u003ch2 id=\"ch2-seq\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#ch2-seq\"\u003e\u003c/a\u003eCH2 seq\u003c/h2\u003e\n\u003cp\u003ethe seq was very clean means my culture is pure\nCH2\u003c/p\u003e\n\u003cblockquote\u003e\n\u003cp\u003eTCATTACAGAGTTGCAAAACTCCCCAAACCATCGTGAACGCTACCTAAACCGTTGCTTCGGCGGGCGGCGCCCTCGCGCGCCCCCCTGGGCCCGCACCGCGGGCGCCCGCCGGAGGTACACCAAACTCTTGATATGTCATGGCCACTCTGAGTCTTCTATACTGAATAAGTCAAAACTTTCAACAACGGATCTCTTGGTTCTGGCATCGATGAAGAACGCAGCGAAATGCGATAAGTAATGTGAATTGCAGAATTCAGTGAATCATCGAATCTTTGAACGCACATTGCGCCCGCCAGGATAGTATCCTGGCGGGCATGCCTGTTCGAGCGTCATTTCAACCCATCAAGCCCACGGCTTGTGTTGGGGACCTGCGGCTGCCCGCAGGCCCTGAAAACCAGTGGCGGGCTCGCTAGTCACACCGAGCGTAGTAGCATACGACCTCGCTCAGGGCGTGCTGCGGGTTCCAGCCGTAAAACGACCTTCGCGACCCCAAGGTTGACCTCGGATCAGGTAGGAGGACCCGCTGAACTTAAGCATATCAATAAGCGGAGGAA\n\u003ca title=\"Private\" href=\"https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html\" target=\"_blank\" class=\"private\"\u003e#NCBI (Private)\u003c/a\u003e BLAST\n\u003cimg src=\"/commons-dws-public/image-1.png\" alt=\"alt text\"\u003e\u003c/p\u003e\n\u003c/blockquote\u003e\n\u003cp\u003eThermothelomyces guttulatus \u003c/p\u003e\n\u003cp\u003eWD query for compounds \u003ca href=\"https://w.wiki/Bnes\"\u003ehttps://w.wiki/Bnes\u003c/a\u003e\u003c/p\u003e\n\u003ch2 id=\"monday\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#monday\"\u003e\u003c/a\u003eMonday\u003c/h2\u003e\n\u003cp\u003eDNA extraction of CH4 and V1 \u003c/p\u003e\n\u003ch2 id=\"active-strains-storage\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#active-strains-storage\"\u003e\u003c/a\u003eactive strains storage\u003c/h2\u003e\n\u003cp\u003eI stored all the active strains in 70% glycerol, each strain has 4 copies, and the tudes are at -80Â°C.\n\u003cimg src=\"/commons-dws-public/conserved%20strains.jpg\" alt=\"alt text\"\u003e\u003c/p\u003e\n\u003ch2 id=\"todays-review-04-11-2024\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#todays-review-04-11-2024\"\u003e\u003c/a\u003etoday's review (04-11-2024)\u003c/h2\u003e\n\u003cp\u003eFinaly i get bands for V1 and CH4 isolates\nsend for sequencing\nall strains stored at -80Â°C.\u003c/p\u003e\n\u003ch2 id=\"tommorow-02-11-2024\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#tommorow-02-11-2024\"\u003e\u003c/a\u003eTommorow 02-11-2024\u003c/h2\u003e\n\u003cp\u003ewrite on my lab note book\nwait for results sequences.\nclean cutures wastes\nanalyses ITS seq of CH4 and 16S for V1. \u003c/p\u003e\n\u003ch2 id=\"we-have-sent-the-sequences-of-v1-and-ch4-to-sequencing-the-results-are-not-reliable-to-be-reextract\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#we-have-sent-the-sequences-of-v1-and-ch4-to-sequencing-the-results-are-not-reliable-to-be-reextract\"\u003e\u003c/a\u003eWe have sent the sequences of V1 and CH4 to sequencing, the results are not reliable, to be reextract.\u003c/h2\u003e\n\u003cp\u003eresults recieved 05-11-2024\nCH4\nCATTACCGAKTGCGGGCTGCCTCCGGGCGCCCAACCTCCCWCCCGTGACTACCTAACACTGTTGCTTCRGCGGGGAGCCCCCCAGGGGGCGAGCCGCCGGGGACCACTGAACTTCATGCCTGASAGKGATGCAGWCTGASCCTGAATACAAATCAGTCAAAACTTTCAACAATGGATCTCTTGGTTCCGGCATCRATGAAGAACGCAGCGAACTGC\u003c/p\u003e\n\u003cp\u003eL'intensitÃ© du signal de votre rÃ©action de sÃ©quenÃ§age est trÃ¨s faible. Par consÃ©quent, le rÃ©sultat de votre sÃ©quenÃ§age doit Ãªtre soigneusement Ã©valuÃ©.\nRÃ©sultat de sÃ©quenÃ§age modÃ©rÃ© ou ambigu. Veuillez vÃ©rifier l'Ã©tat de votre rÃ©action, qui sera mise Ã  jour dans la zone de tÃ©lÃ©chargement dans le courant de la matinÃ©e (jours ouvrables uniquement).\nEconomy Run (Tube)\n13787654 94799985\nV1\nNNNNN\u003c/p\u003e\n\u003ch2 id=\"today-06-11-2024\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#today-06-11-2024\"\u003e\u003c/a\u003eToday 06-11-2024\u003c/h2\u003e\n\u003cp\u003e-DNA extraction of CH4 AND V1\u003c/p\u003e\n\u003ch2 id=\"tomorrow-07-11-2024\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#tomorrow-07-11-2024\"\u003e\u003c/a\u003eTomorrow 07-11-2024\u003c/h2\u003e\n\u003cp\u003ePCR, agarose gel and purification.\u003c/p\u003e\n\u003ch2 id=\"revievs-07-11-2024\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#revievs-07-11-2024\"\u003e\u003c/a\u003eReviev's 07-11-2024\u003c/h2\u003e\n\u003chr\u003e\n\u003cstrong\u003eChildren\u003c/strong\u003e\n\u003col\u003e\n\u003cli\u003e\u003ca href=\"/commons-dws-public/notes/kvtssreflme8xid39zy3t0n\"\u003eStreptomyces lusitanus\u003c/a\u003e\u003c/li\u003e\n\u003c/ol\u003e","noteIndex":{"id":"QvYK9hGbCvPpEfSRYhV8j","title":"Welcome to the COMMONS Lab Open Dendron","desc":"","updated":1693056226815,"created":1630130450048,"custom":{"nav_order":0,"permalink":"/"},"fname":"root","type":"note","vault":{"fsPath":"vault"},"contentHash":"20f67df3dd870fc450a162c202c9ff41","links":[{"type":"wiki","from":{"fname":"root","id":"QvYK9hGbCvPpEfSRYhV8j","vaultName":"vault"},"value":"open-notebook.commons.setup","position":{"start":{"line":43,"column":118,"offset":1871},"end":{"line":43,"column":149,"offset":1902},"indent":[]},"xvault":false,"sameFile":false,"to":{"fname":"open-notebook.commons.setup"}}],"anchors":{"what-is-this-note":{"type":"header","text":"What is this note","value":"what-is-this-note","line":16,"column":0,"depth":2},"tutorial":{"type":"header","text":"Tutorial","value":"tutorial","line":20,"column":0,"depth":2},"dendron":{"type":"header","text":"Dendron.","value":"dendron","line":22,"column":0,"depth":3},"what-is-dendron-":{"type":"header","text":"What is Dendron ?","value":"what-is-dendron-","line":24,"column":0,"depth":4},"where-to-get-more-info-on-dendron-":{"type":"header","text":"Where to get more info on Dendron ?","value":"where-to-get-more-info-on-dendron-","line":30,"column":0,"depth":4},"concrete-steps-to-access-and-contribute-to-the-dbgi-dendron":{"type":"header","text":"Concrete steps to access and contribute to the DBGI Dendron","value":"concrete-steps-to-access-and-contribute-to-the-dbgi-dendron","line":35,"column":0,"depth":4},"questions-comments-suggestions-":{"type":"header","text":"Questions, comments, suggestions ?","value":"questions-comments-suggestions-","line":52,"column":0,"depth":2}},"children":["5o0tvzf4l6t6moau7en1v48","e4ul30admmxm9ilxfq275cq","h78qbvh1fd9bwcykmcm9uo3","6ooxuf27zzf3grb14o18qf2","tiqo5upbg6224t7i8p87iok","ltq8wcrabmq6mlescrdi4m0","bdq3suiz8yriznljea2zpug","ih6vyup3yre4m9woc9ldl7k","vDTgZL9UHWqYtBFdtD3vK","69ekb2qhuwukr0jwrzy6tnw","373694o1o6ocqohko4um7c9","jldFUSJGjDf1mFH8c2yUI","dh2qp7w4tl6otf89xrmxitv","2rvdf4t5qnx5hqjj29bojcc","stj9lq0lxhfuk5ntomiozz0","gvdyfgvq0z08fcfaqzmhtjo","2rvclk21kfedpvxuzjpizi7","dcwgamgyghrlyau1avlh6ug","432qdutuo1i8h5pcuh39ytt","2npxodkvs5jk6p1eksi551b","h81rw16zzgh9kix8ti1mbam","lco50o42dfeph5i5f2k981w","46d1odkovrpdi6g16992phy","s9gzk7nphtjzmpdqvk5y4vj","82ra6p4ykf814hp8yfw4wp6","hpnaglyhnb48vuiniqecnqf","y7nk7xmiht44neo0q7cl5cs","2cxr093kvrg4jojokwgqrbj","8vfu5h3617jbzy78xr2hlo1","w75krbmkkvla3hwd66hn5bh","40po5od17ekcr1e77dluxao","4pqppog2u66rx1tkc8jy5v9","ektni5nfulyro7dkrbtek7j","vfsp2aci6vy7kg2jsyy8tzu","0wm4geq40t6nscn88rcwu47","ib4jmxy1x3l1r0ewe6hln4j","0h57aw0be2jnc3dxcpn3ajn","4789z8f1dqdh5k3nmgr1m4z","p5dszyuhj1klgje0brtv0xj","sk4m9gq7vz2t9phcxlxvqpw","43p60vv8btyhaclugl93jok","zhc9hhpaobkbo4e2votv6wr","ylulfzi7ra5yv6ja7j78t6e","wj0i9czxvt4v0b674nb8501","ovslc641d0h00s16tf6kl8w","4zmlbmcp6q4q69ycvhavzr6","59ftsVxFXXM9bB43vfKwK","nmnpsy2e1rwqnqgurogi2u6","6b5uf8hv44mbgircddcb46i","s9j6c26u6qgl5dvnqv8opam","je0ksq927btmorptaje6a4p","xm4a1smh9gcahxda5bssxfi","3cgqlppr4es3iuoa2o3mhn8","d6edcr5j5kjgdt791uujpn0","zfu2gniwqhgo1j44r3yfx7y","gap1pjhvt94m5izg0zlef0w","o3y0vguickctv1yx4z42qev","g78c0e4ts7wgkof1ykkgzbl","hsqldiv7revakz0fqay51eu","99wwtyhexdqu5bxrtgn9whw","m8e37qw6n4i3cfv9ohq7e82","ds1s2fjqhwvw1zqst8l4z4f","ec4pkybo5iwljo3zdr9894x","cbikmu2vjnxulf5tf9xfist","1icb0ka0yjwpoerg4zym9la"],"parent":null,"data":{},"body":"\n![](/assets/images/allard-lab.jpg)\n\nWelcome the the COMMONS Lab Open Dendron !\n\nIn the [COMMONS Lab](https://www.unifr.ch/bio/en/groups/allard/) we intent to follow the [Open Notebook Science](https://en.wikipedia.org/wiki/Open-notebook_science) approach to document our research.\n\nFor this we use the [Dendron](https://www.dendron.so/) system as a mean to efficiently capture notes and publish them.\n\n## What is this note\n\nThis note is a succinct tutorial note aiming to get you started in the use of the COMMONS Lab Dendron.\n\n## Tutorial\n\n### Dendron. \n\n#### What is Dendron ?\n\n\u003e Dendron is an open-source, local-first, markdown-based, note-taking tool. Think of it as a cache for everything that you care about - if you've spent more then five minutes solving a problem, you should never spent any more time solving the same exact problem.\n\u003e \n\u003e Dendron is a knowledge base built by and for developers and integrates natively with IDEs like VS Code and VSCodium.\n\n#### Where to get more info on Dendron ?\n\n- You can get more information in the Dendron system at the official website www.dendron.so\n- All the Dendron documentation is hosted here https://wiki.dendron.so/. It is, obviously, a Dendron itself.\n\n#### Concrete steps to access and contribute to the DBGI Dendron\n\n1. Install [VSCode](https://code.visualstudio.com/download)\n2. Install [Dendron](https://marketplace.visualstudio.com/items?itemName=dendron.dendron) from the VSCode marketplace \n3. Clone the COMMONS Dendron repo\n\n    In your terminal\n    ```\n    git clone https://github.com/commons-research/commons-dws-public.git\n    ```\n4. In VSCode open (File/Open) the cloned repository.\n5. Voila ! You should now be in the COMMONS Lab Open Dendron.\n\nFeel free to explore and contribute.\nA first, important step could be to configure VSCode so that you can easily add a daily note. See steps described at [[open-notebook.commons.setup]]\n\n\n## Questions, comments, suggestions ?\n\nFeel free to contribute here https://github.com/orgs/commons-research/discussions\n\n"},"collectionChildren":null,"customHeadContent":null,"config":{"version":5,"dev":{"enablePreviewV2":true},"commands":{"lookup":{"note":{"selectionMode":"extract","confirmVaultOnCreate":false,"leaveTrace":false,"bubbleUpCreateNew":true,"vaultSelectionModeOnCreate":"smart","fuzzThreshold":0.2}},"insertNote":{"initialValue":"templates"},"insertNoteLink":{"aliasMode":"none","enableMultiSelect":false},"insertNoteIndex":{"enableMarker":false},"randomNote":{},"copyNoteLink":{"aliasMode":"title"},"templateHierarchy":"template"},"workspace":{"vaults":[{"fsPath":"vault"}],"journal":{"dailyDomain":"daily","name":"journal","dateFormat":"y.MM.dd","addBehavior":"childOfDomain"},"scratch":{"name":"scratch","dateFormat":"y.MM.dd.HHmmss","addBehavior":"asOwnDomain"},"graph":{"zoomSpeed":1,"createStub":false},"enableAutoCreateOnDefinition":false,"enableXVaultWikiLink":false,"enableRemoteVaultInit":true,"workspaceVaultSyncMode":"noCommit","enableAutoFoldFrontmatter":false,"maxPreviewsCached":10,"maxNoteLength":204800,"task":{"name":"","dateFormat":"","addBehavior":"childOfCurrent","statusSymbols":{"":" ","wip":"w","done":"x","assigned":"a","moved":"m","blocked":"b","delegated":"l","dropped":"d","pending":"y"},"prioritySymbols":{"H":"high","M":"medium","L":"low"},"todoIntegration":false,"createTaskSelectionType":"selection2link","taskCompleteStatus":["done","x"]},"enableUserTags":true,"enableHashTags":true,"dendronVersion":"0.112.0","enableEditorDecorations":true,"enableFullHierarchyNoteTitle":false},"preview":{"enableFMTitle":true,"enableNoteTitleForLink":true,"enableMermaid":true,"enablePrettyRefs":true,"enableKatex":true,"automaticallyShowPreview":false,"enableFrontmatterTags":true,"enableHashesForFMTags":false},"publishing":{"enableFMTitle":true,"enableNoteTitleForLink":true,"enableMermaid":true,"enablePrettyRefs":true,"enableKatex":true,"assetsPrefix":"/commons-dws-public","copyAssets":true,"siteHierarchies":["root"],"enableSiteLastModified":true,"siteRootDir":"docs","siteUrl":"https://commons-research.github.io","enableFrontmatterTags":true,"enableHashesForFMTags":false,"enableRandomlyColoredTags":true,"duplicateNoteBehavior":{"action":"useVault","payload":["vault"]},"writeStubs":false,"seo":{"title":"COMMONS DWS Public","description":"Personal knowledge space"},"github":{"enableEditLink":true,"editLinkText":"Click here to edit this page on Github !","editBranch":"main","editViewMode":"edit","editRepository":"https://github.com/commons-research/commons-dws-public"},"enablePrettyLinks":true,"enableTaskNotes":true,"siteFaviconPath":"favicon.ico","siteIndex":"root","searchMode":"lookup"}}},"__N_SSG":true},"page":"/notes/[id]","query":{"id":"a0iu9pk5ktxoo6h7y7gmm6e"},"buildId":"KWnqrj5vUmxn1r58uXWxO","assetPrefix":"/commons-dws-public","isFallback":false,"gsp":true,"scriptLoader":[]}</script></body></html>