<!DOCTYPE html><html><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width"/><link rel="icon" href="/commons-dws-public/favicon.ico"/><title>Papers</title><meta name="robots" content="index,follow"/><meta name="googlebot" content="index,follow"/><meta name="description" content="Personal knowledge space"/><meta property="og:title" content="Papers"/><meta property="og:description" content="Personal knowledge space"/><meta property="og:url" content="https://commons-research.github.io/commons-dws-public/notes/loztevovn8nanzdk302b756/"/><meta property="og:type" content="article"/><meta property="article:published_time" content="2/29/2024"/><meta property="article:modified_time" content="4/11/2024"/><link rel="canonical" href="https://commons-research.github.io/commons-dws-public/notes/loztevovn8nanzdk302b756/"/><meta name="next-head-count" content="14"/><link rel="preload" href="/commons-dws-public/_next/static/css/47f58a23998994fc.css" as="style"/><link rel="stylesheet" href="/commons-dws-public/_next/static/css/47f58a23998994fc.css" data-n-g=""/><noscript data-n-css=""></noscript><script defer="" nomodule="" src="/commons-dws-public/_next/static/chunks/polyfills-c67a75d1b6f99dc8.js"></script><script src="/commons-dws-public/_next/static/chunks/webpack-18f1eb2dd0aac0df.js" defer=""></script><script src="/commons-dws-public/_next/static/chunks/framework-28c999baf2863c3d.js" defer=""></script><script src="/commons-dws-public/_next/static/chunks/main-1bd061c8c8d18148.js" defer=""></script><script src="/commons-dws-public/_next/static/chunks/pages/_app-8bc9d81129a787a1.js" defer=""></script><script src="/commons-dws-public/_next/static/chunks/935-4dee79e80b8641c6.js" defer=""></script><script src="/commons-dws-public/_next/static/chunks/6-50972def09142ee2.js" defer=""></script><script src="/commons-dws-public/_next/static/chunks/pages/notes/%5Bid%5D-78d472fa3b924116.js" defer=""></script><script src="/commons-dws-public/_next/static/ilh4ComeLwDbrM8zcKyPY/_buildManifest.js" defer=""></script><script src="/commons-dws-public/_next/static/ilh4ComeLwDbrM8zcKyPY/_ssgManifest.js" defer=""></script></head><body><div id="__next" data-reactroot=""><section class="ant-layout" style="width:100%;min-height:100%"><header class="ant-layout-header" style="position:fixed;isolation:isolate;z-index:1;width:100%;border-bottom:1px solid #d4dadf;height:64px;padding:0 24px 0 2px"><div class="ant-row ant-row-center" style="max-width:992px;justify-content:space-between;margin:0 auto"><div style="display:flex" class="ant-col ant-col-xs-20 ant-col-sm-4"></div><div class="ant-col gutter-row ant-col-xs-0 ant-col-sm-20 ant-col-md-20 ant-col-lg-19"><div class="ant-select ant-select-lg ant-select-auto-complete ant-select-single ant-select-allow-clear ant-select-show-search" style="width:100%"><div class="ant-select-selector"><span class="ant-select-selection-search"><input type="search" autoComplete="off" class="ant-select-selection-search-input" role="combobox" aria-haspopup="listbox" aria-owns="undefined_list" aria-autocomplete="list" aria-controls="undefined_list" aria-activedescendant="undefined_list_0" value=""/></span><span class="ant-select-selection-placeholder">For full text search please use the &#x27;?&#x27; prefix. e.g. ? Onboarding</span></div></div></div><div style="display:none;align-items:center;justify-content:center" class="ant-col ant-col-xs-4 ant-col-sm-4 ant-col-md-0 ant-col-lg-0"><span role="img" aria-label="menu" style="font-size:24px" tabindex="-1" class="anticon anticon-menu"><svg viewBox="64 64 896 896" focusable="false" data-icon="menu" width="1em" height="1em" fill="currentColor" aria-hidden="true"><path d="M904 160H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0 624H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0-312H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8z"></path></svg></span></div></div></header><section class="ant-layout" style="margin-top:64px;display:flex;flex-direction:row"><div class="site-layout-sidebar" style="flex:0 0 auto;width:calc(max((100% - 992px) / 2, 0px) + 200px);min-width:200px;padding-left:calc((100% - 992px) / 2)"><aside class="ant-layout-sider ant-layout-sider-dark" style="position:fixed;overflow:auto;height:calc(100vh - 64px);background-color:transparent;flex:0 0 200px;max-width:200px;min-width:200px;width:200px"><div class="ant-layout-sider-children"></div></aside></div><main class="ant-layout-content side-layout-main" style="max-width:1200px;min-width:0;display:block"><div style="padding:0 24px"><div class="main-content" role="main"><div class="ant-row"><div class="ant-col ant-col-24"><div class="ant-row" style="margin-left:-10px;margin-right:-10px"><div style="padding-left:10px;padding-right:10px" class="ant-col ant-col-xs-24 ant-col-md-18"><div><h1 id="papers"><a aria-hidden="true" class="anchor-heading icon-link" href="#papers"></a>Papers</h1>
<h2 id="papers-1"><a aria-hidden="true" class="anchor-heading icon-link" href="#papers-1"></a>Paper's</h2>
<h3 id="lotus"><a aria-hidden="true" class="anchor-heading icon-link" href="#lotus"></a><a href="https://doi.org/10.7554/eLife.70780">LOTUS</a></h3>
<p><a href="https://lotus.naturalproducts.net">LOTUS (LOpen Knowledge Management in Natural Products Research)</a> is a database of over 750'000+ referenced structure-organism pairs. A structure-organims pairs shows the relationship between organisms and molecules.
With the increasing technology, the data for chemical compounds (exp. mass specrometry) and organisms (exp. sequencing machines) also increases exponentialy. To link the data will be more and more beneficial.<br>
The idea of LOTUS is to interconnect the three central objects of the living (metabolites, proteins and genes).  </p>
<p>Step 1 - Data harmonization:<br>
The information comes from different sources, which has to be harmonized/standardized for building the new DB.
The scripts involves a comparison with the gathered data to ensure no duplication (SSOT).  </p>
<p>Step 2 - Precessing and Validation:<br>
The tree central objects where taken togheter. Mostly the references was particularly challenging (poor quality).
An unit test checked the code after each changes for the same results.</p>
<p>Step 3 - Dissemination:<br>
The researchdata should be accessible following the FAIR principles. SPARQL are generic queries to return specific "tables".
To be find, they have to be in a specific format.</p>
<p>For the biologically interpretation of chemicals is used TMAP. The biological specificity score is calculated as a Jensen-Shannon divergence.
1 means very specific chemical compound and 0 that they are ubquitous.
The same is done in the opposit way (chemically interpretation of biology).
Aim to make more and better data available.</p>
<h4 id="links"><a aria-hidden="true" class="anchor-heading icon-link" href="#links"></a>Link's</h4>
<p><a href="/commons-dws-public/">KNAPSAck</a> 50'000+ structures and 100'000+ structure-organism pairs<br>
<a href="/commons-dws-public/">NAPRA-LERT dataset</a> well documented database but not available for free<br>
<a href="/commons-dws-public/">NPAtlas</a> specificly for microbial NPs.<br>
<a href="/commons-dws-public/">COCONUT</a> COlleCtion of Open NatUral products (similar to LOTUS)<br>
<a href="https://naturalproducts.net/">natural products online</a> - Host for Lotus, Coconut ...  </p>
<h3 id="mines-open-access-databases-of-computationally-predicted-enzyme-promiscuity-products-for-untargeted-metabolomics"><a aria-hidden="true" class="anchor-heading icon-link" href="#mines-open-access-databases-of-computationally-predicted-enzyme-promiscuity-products-for-untargeted-metabolomics"></a><a href="https://doi.org/10.1186/s13321-015-0087-1">MINEs: open access databases of computationally predicted enzyme promiscuity products for untargeted metabolomics</a></h3>
<p>MINEs are an extension for metabolites database, which have been not observed, but very likely can occure. Based on the Biochemical Network Integrated Computational Explorer and enzyme reaction rules it will be calculated. The website is userfriendly and also available as an API: <a href="https://minedatabase.mcs.anl.gov/#/home">https://minedatabase.mcs.anl.gov/#/home</a></p>
<p>This database includes structures for use in untargeted metabolomics. This resulting compounds are rarely found in PubChem but are structurally similar to natural products.</p>
<p>As an input you can give an MS/MS file, structure, name.... As an output you will get information about the reaction, validated compounds...</p>
<h3 id="molecular-structure-discovery-for-untargeted-metabolomics-using-biotransformation-rules-and-global-molecular-networking"><a aria-hidden="true" class="anchor-heading icon-link" href="#molecular-structure-discovery-for-untargeted-metabolomics-using-biotransformation-rules-and-global-molecular-networking"></a><a href="https://doi.org/10.1101/2024.02.04.578795">Molecular structure discovery for untargeted metabolomics using biotransformation rules and global molecular networking</a></h3>
<p>A big part of the MS/MS spectras are unidentified (exp. GNPS/MassIVE).
For this problem they developed BAM (Biotransformation-based Annotation Method).
It has the potential to identified new molecules based on previously unknown MS/MS spectra.
The predictionrate for the right assignment is 24.x %.  </p>
<p>BAM concist of three steps:</p>
<ol>
<li>
<p>determine an annotated anchor spectrum based on high spectral similarity.  </p>
</li>
<li>
<p>Apply biotranformation rules that match the observed mass difference and generate candidate structures<br>
The rules are sourced from RetroRules, KEGG or Metacyc. Out of that, PROXIMALE 2 was employed.</p>
</li>
<li>
<p>Rank the derivatives based on the likelihods of the applied biotransformation.
GNN-SOM predicts the likelyhood of each atom in a molecule.</p>
</li>
</ol>
<h4 id="coding"><a aria-hidden="true" class="anchor-heading icon-link" href="#coding"></a>Coding</h4>
<p><a href="https://github.com/HassounLab/BAM">Github - HassounLab</a>
PROXIMAL2<br>
GNN-SOM tool (graph neural network-based tool)</p>
<h3 id="pickaxe"><a aria-hidden="true" class="anchor-heading icon-link" href="#pickaxe"></a><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-023-05149-8">Pickaxe...</a></h3>
<ul>
<li>prepackaged ruleset (JN1224min - metacyc) with UniProt IDs of enzyms</li>
<li>rules provided on Pickaxe do not utilize stereochemistry</li>
<li>RDKit: Cheminformatics program applies the SMARTS rules.</li>
<li>compounds and reactions can be filtered in the same run.</li>
</ul>
<p>Filters:</p>
<ul>
<li>
<p>Similarity Filtering (Tanimoto):
Info: filters out the samples, which are not higher than the given similarity treshhold. Similarity is calculated with the Tanimoto distance of the targets and new made molecules.
Input:<br>
cutoff_fingerprint_method = "Morgan"
cutoff_similarity_method = "Tanimoto"
similarity_threshold = [0, 0.9]
Output:  </p>
</li>
<li>
<p>Similarity Sampling Filter:
Info: samples a specific amount of a given samplespace (made by the simlarity filter). The similarity filter has to be activated!!!
Input:
cutoff_fingerprint_method = "Morgan"
cutoff_similarity_method = "Tanimoto"
similarity_threshold = [0, 0.9]
Output:
n-samples</p>
</li>
<li>
<p>Metabolomics filter:
Input: list of experimental mass-to-charge (m/z), list of possible adducts
Output: compares the two lists by the m/z values and give back all compounds with a specific treshhold</p>
</li>
<li>
<p>Thermodynamic filter:
Info: eQuilator (module) biochemical thermodynamic calculator
Input: </p>
</li>
<li>
<p>Feasibility filter:
Info: DeepRFC (<a href="https://%E2%80%8Bdoi.%E2%80%8Borg/%E2%80%8B10.%E2%80%8B1002/%E2%80%8Bbiot.%E2%80%8B20200%E2%80%8B0605">https://​doi.​org/​10.​1002/​biot.​20200​0605</a>) DeepRFC utilizes substrate-product pairs within KEGG reactions to train a deep neural net to predict a reaction’s feasibility.</p>
</li>
</ul>
<p><img src="/commons-dws-public/expanded-np-chemspace.distancemeters_chemical_moleculs.png" alt=" distancemeters "><br>
<em>distancemeters <a href="https://www.sciencedirect.com/science/article/pii/S1046202314002631?via%3Dihub">https://www.sciencedirect.com/science/article/pii/S1046202314002631?via%3Dihub</a></em></p>
<h3 id="a-deep-learning-approach-to-evaluate-the-feasibility-of-enzymatic-reactions-generated-by-retrobiosynthesis"><a aria-hidden="true" class="anchor-heading icon-link" href="#a-deep-learning-approach-to-evaluate-the-feasibility-of-enzymatic-reactions-generated-by-retrobiosynthesis"></a><a href="https://analyticalsciencejournals.onlinelibrary.wiley.com/doi/10.1002/biot.202000605">A deep learning approach to evaluate the feasibility of enzymatic reactions generated by retrobiosynthesis</a></h3>
<p>enzymatic reactions generated by retrobiosynthesis
DOI: 10.1002/biot.202000605</p>
<h3 id="rdp---filter"><a aria-hidden="true" class="anchor-heading icon-link" href="#rdp---filter"></a><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4920114/">RDP - Filter</a></h3>
<p>10.1093/bioinformatics/btw096</p>
<p><a href="/commons-dws-public/notes/3ca85tg6eqf78kgql1mj3vl">Abbreviations</a></p>
<hr>
<strong>Backlinks</strong>
<ul>
<li><a href="/commons-dws-public/notes/0vbuzuauxdhmsbckhc0pye0">Information</a></li>
<li><a href="/commons-dws-public/notes/qy0iqvyrj0me4co8u8d25nh">2024-02-21</a></li>
<li><a href="/commons-dws-public/notes/oxoker8doqq8gxa0e16wnp1">2024-02-22</a></li>
<li><a href="/commons-dws-public/notes/wn10dc3l535y60u8yb9iq2d">2024-02-23</a></li>
</ul></div></div><div style="padding-left:10px;padding-right:10px" class="ant-col ant-col-xs-0 ant-col-md-6"><div><div class=""><div class="ant-anchor-wrapper dendron-toc" style="max-height:calc(100vh - 64px);z-index:1"><div class="ant-anchor"><div class="ant-anchor-ink"><span class="ant-anchor-ink-ball"></span></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#papers" title="Paper&#x27;s">Paper&#x27;s</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#lotus" title="LOTUS">LOTUS</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#links" title="Link&#x27;s">Link&#x27;s</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#mines-open-access-databases-of-computationally-predicted-enzyme-promiscuity-products-for-untargeted-metabolomics" title="MINEs: open access databases of computationally predicted enzyme promiscuity products for untargeted metabolomics">MINEs: open access databases of computationally predicted enzyme promiscuity products for untargeted metabolomics</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#molecular-structure-discovery-for-untargeted-metabolomics-using-biotransformation-rules-and-global-molecular-networking" title="Molecular structure discovery for untargeted metabolomics using biotransformation rules and global molecular networking">Molecular structure discovery for untargeted metabolomics using biotransformation rules and global molecular networking</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#coding" title="Coding">Coding</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#pickaxe" title="Pickaxe...">Pickaxe...</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#a-deep-learning-approach-to-evaluate-the-feasibility-of-enzymatic-reactions-generated-by-retrobiosynthesis" title="A deep learning approach to evaluate the feasibility of enzymatic reactions generated by retrobiosynthesis">A deep learning approach to evaluate the feasibility of enzymatic reactions generated by retrobiosynthesis</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#rdp---filter" title="RDP - Filter">RDP - Filter</a></div></div></div></div></div></div></div></div></div></div></div><div class="ant-divider ant-divider-horizontal" role="separator"></div><footer class="ant-layout-footer" style="padding:0 24px 24px"></footer></main></section></section></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{"note":{"id":"loztevovn8nanzdk302b756","title":"Papers","desc":"","updated":1712844671561,"created":1709212733420,"custom":{},"fname":"expanded-np-chemspace.information.papers","type":"note","vault":{"fsPath":"vault"},"contentHash":"676e2c3de54bef67cfa72ccf7412d2fe","links":[{"type":"wiki","from":{"fname":"expanded-np-chemspace.information.papers","id":"loztevovn8nanzdk302b756","vaultName":"vault"},"value":"expanded_np_chemspace.abbreviations","position":{"start":{"line":116,"column":1,"offset":6086},"end":{"line":116,"column":43,"offset":6128},"indent":[]},"xvault":false,"sameFile":false,"to":{"fname":"expanded_np_chemspace.abbreviations"}},{"from":{"id":"0vbuzuauxdhmsbckhc0pye0","fname":"expanded-np-chemspace.information","vaultName":"vault"},"type":"backlink","position":{"start":{"line":10,"column":1,"offset":147},"end":{"line":10,"column":45,"offset":191},"indent":[]},"value":"expanded-np-chemspace.information.papers"},{"from":{"id":"qy0iqvyrj0me4co8u8d25nh","fname":"open-notebook.commons.pamrein.2024.02.21","vaultName":"vault"},"type":"backlink","position":{"start":{"line":9,"column":1,"offset":235},"end":{"line":9,"column":45,"offset":279},"indent":[]},"value":"expanded-np-chemspace.information.papers"},{"from":{"id":"oxoker8doqq8gxa0e16wnp1","fname":"open-notebook.commons.pamrein.2024.02.22","vaultName":"vault"},"type":"backlink","position":{"start":{"line":15,"column":1,"offset":330},"end":{"line":15,"column":45,"offset":374},"indent":[]},"value":"expanded-np-chemspace.information.papers"},{"from":{"id":"wn10dc3l535y60u8yb9iq2d","fname":"open-notebook.commons.pamrein.2024.02.23","vaultName":"vault"},"type":"backlink","position":{"start":{"line":14,"column":1,"offset":211},"end":{"line":14,"column":45,"offset":255},"indent":[]},"value":"expanded-np-chemspace.information.papers"}],"anchors":{"papers":{"type":"header","text":"Paper's","value":"papers","line":8,"column":0,"depth":2},"lotus":{"type":"header","text":"LOTUS","value":"lotus","line":9,"column":0,"depth":3},"links":{"type":"header","text":"Link's","value":"links","line":33,"column":0,"depth":4},"mines-open-access-databases-of-computationally-predicted-enzyme-promiscuity-products-for-untargeted-metabolomics":{"type":"header","text":"MINEs: open access databases of computationally predicted enzyme promiscuity products for untargeted metabolomics","value":"mines-open-access-databases-of-computationally-predicted-enzyme-promiscuity-products-for-untargeted-metabolomics","line":41,"column":0,"depth":3},"molecular-structure-discovery-for-untargeted-metabolomics-using-biotransformation-rules-and-global-molecular-networking":{"type":"header","text":"Molecular structure discovery for untargeted metabolomics using biotransformation rules and global molecular networking","value":"molecular-structure-discovery-for-untargeted-metabolomics-using-biotransformation-rules-and-global-molecular-networking","line":50,"column":0,"depth":3},"coding":{"type":"header","text":"Coding","value":"coding","line":68,"column":0,"depth":4},"pickaxe":{"type":"header","text":"Pickaxe...","value":"pickaxe","line":74,"column":0,"depth":3},"a-deep-learning-approach-to-evaluate-the-feasibility-of-enzymatic-reactions-generated-by-retrobiosynthesis":{"type":"header","text":"A deep learning approach to evaluate the feasibility of enzymatic reactions generated by retrobiosynthesis","value":"a-deep-learning-approach-to-evaluate-the-feasibility-of-enzymatic-reactions-generated-by-retrobiosynthesis","line":113,"column":0,"depth":3},"rdp---filter":{"type":"header","text":"RDP - Filter","value":"rdp---filter","line":117,"column":0,"depth":3}},"children":[],"parent":"0vbuzuauxdhmsbckhc0pye0","data":{}},"body":"\u003ch1 id=\"papers\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#papers\"\u003e\u003c/a\u003ePapers\u003c/h1\u003e\n\u003ch2 id=\"papers-1\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#papers-1\"\u003e\u003c/a\u003ePaper's\u003c/h2\u003e\n\u003ch3 id=\"lotus\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#lotus\"\u003e\u003c/a\u003e\u003ca href=\"https://doi.org/10.7554/eLife.70780\"\u003eLOTUS\u003c/a\u003e\u003c/h3\u003e\n\u003cp\u003e\u003ca href=\"https://lotus.naturalproducts.net\"\u003eLOTUS (LOpen Knowledge Management in Natural Products Research)\u003c/a\u003e is a database of over 750'000+ referenced structure-organism pairs. A structure-organims pairs shows the relationship between organisms and molecules.\nWith the increasing technology, the data for chemical compounds (exp. mass specrometry) and organisms (exp. sequencing machines) also increases exponentialy. To link the data will be more and more beneficial.\u003cbr\u003e\nThe idea of LOTUS is to interconnect the three central objects of the living (metabolites, proteins and genes).  \u003c/p\u003e\n\u003cp\u003eStep 1 - Data harmonization:\u003cbr\u003e\nThe information comes from different sources, which has to be harmonized/standardized for building the new DB.\nThe scripts involves a comparison with the gathered data to ensure no duplication (SSOT).  \u003c/p\u003e\n\u003cp\u003eStep 2 - Precessing and Validation:\u003cbr\u003e\nThe tree central objects where taken togheter. Mostly the references was particularly challenging (poor quality).\nAn unit test checked the code after each changes for the same results.\u003c/p\u003e\n\u003cp\u003eStep 3 - Dissemination:\u003cbr\u003e\nThe researchdata should be accessible following the FAIR principles. SPARQL are generic queries to return specific \"tables\".\nTo be find, they have to be in a specific format.\u003c/p\u003e\n\u003cp\u003eFor the biologically interpretation of chemicals is used TMAP. The biological specificity score is calculated as a Jensen-Shannon divergence.\n1 means very specific chemical compound and 0 that they are ubquitous.\nThe same is done in the opposit way (chemically interpretation of biology).\nAim to make more and better data available.\u003c/p\u003e\n\u003ch4 id=\"links\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#links\"\u003e\u003c/a\u003eLink's\u003c/h4\u003e\n\u003cp\u003e\u003ca href=\"/commons-dws-public/\"\u003eKNAPSAck\u003c/a\u003e 50'000+ structures and 100'000+ structure-organism pairs\u003cbr\u003e\n\u003ca href=\"/commons-dws-public/\"\u003eNAPRA-LERT dataset\u003c/a\u003e well documented database but not available for free\u003cbr\u003e\n\u003ca href=\"/commons-dws-public/\"\u003eNPAtlas\u003c/a\u003e specificly for microbial NPs.\u003cbr\u003e\n\u003ca href=\"/commons-dws-public/\"\u003eCOCONUT\u003c/a\u003e COlleCtion of Open NatUral products (similar to LOTUS)\u003cbr\u003e\n\u003ca href=\"https://naturalproducts.net/\"\u003enatural products online\u003c/a\u003e - Host for Lotus, Coconut ...  \u003c/p\u003e\n\u003ch3 id=\"mines-open-access-databases-of-computationally-predicted-enzyme-promiscuity-products-for-untargeted-metabolomics\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#mines-open-access-databases-of-computationally-predicted-enzyme-promiscuity-products-for-untargeted-metabolomics\"\u003e\u003c/a\u003e\u003ca href=\"https://doi.org/10.1186/s13321-015-0087-1\"\u003eMINEs: open access databases of computationally predicted enzyme promiscuity products for untargeted metabolomics\u003c/a\u003e\u003c/h3\u003e\n\u003cp\u003eMINEs are an extension for metabolites database, which have been not observed, but very likely can occure. Based on the Biochemical Network Integrated Computational Explorer and enzyme reaction rules it will be calculated. The website is userfriendly and also available as an API: \u003ca href=\"https://minedatabase.mcs.anl.gov/#/home\"\u003ehttps://minedatabase.mcs.anl.gov/#/home\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eThis database includes structures for use in untargeted metabolomics. This resulting compounds are rarely found in PubChem but are structurally similar to natural products.\u003c/p\u003e\n\u003cp\u003eAs an input you can give an MS/MS file, structure, name.... As an output you will get information about the reaction, validated compounds...\u003c/p\u003e\n\u003ch3 id=\"molecular-structure-discovery-for-untargeted-metabolomics-using-biotransformation-rules-and-global-molecular-networking\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#molecular-structure-discovery-for-untargeted-metabolomics-using-biotransformation-rules-and-global-molecular-networking\"\u003e\u003c/a\u003e\u003ca href=\"https://doi.org/10.1101/2024.02.04.578795\"\u003eMolecular structure discovery for untargeted metabolomics using biotransformation rules and global molecular networking\u003c/a\u003e\u003c/h3\u003e\n\u003cp\u003eA big part of the MS/MS spectras are unidentified (exp. GNPS/MassIVE).\nFor this problem they developed BAM (Biotransformation-based Annotation Method).\nIt has the potential to identified new molecules based on previously unknown MS/MS spectra.\nThe predictionrate for the right assignment is 24.x %.  \u003c/p\u003e\n\u003cp\u003eBAM concist of three steps:\u003c/p\u003e\n\u003col\u003e\n\u003cli\u003e\n\u003cp\u003edetermine an annotated anchor spectrum based on high spectral similarity.  \u003c/p\u003e\n\u003c/li\u003e\n\u003cli\u003e\n\u003cp\u003eApply biotranformation rules that match the observed mass difference and generate candidate structures\u003cbr\u003e\nThe rules are sourced from RetroRules, KEGG or Metacyc. Out of that, PROXIMALE 2 was employed.\u003c/p\u003e\n\u003c/li\u003e\n\u003cli\u003e\n\u003cp\u003eRank the derivatives based on the likelihods of the applied biotransformation.\nGNN-SOM predicts the likelyhood of each atom in a molecule.\u003c/p\u003e\n\u003c/li\u003e\n\u003c/ol\u003e\n\u003ch4 id=\"coding\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#coding\"\u003e\u003c/a\u003eCoding\u003c/h4\u003e\n\u003cp\u003e\u003ca href=\"https://github.com/HassounLab/BAM\"\u003eGithub - HassounLab\u003c/a\u003e\nPROXIMAL2\u003cbr\u003e\nGNN-SOM tool (graph neural network-based tool)\u003c/p\u003e\n\u003ch3 id=\"pickaxe\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#pickaxe\"\u003e\u003c/a\u003e\u003ca href=\"https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-023-05149-8\"\u003ePickaxe...\u003c/a\u003e\u003c/h3\u003e\n\u003cul\u003e\n\u003cli\u003eprepackaged ruleset (JN1224min - metacyc) with UniProt IDs of enzyms\u003c/li\u003e\n\u003cli\u003erules provided on Pickaxe do not utilize stereochemistry\u003c/li\u003e\n\u003cli\u003eRDKit: Cheminformatics program applies the SMARTS rules.\u003c/li\u003e\n\u003cli\u003ecompounds and reactions can be filtered in the same run.\u003c/li\u003e\n\u003c/ul\u003e\n\u003cp\u003eFilters:\u003c/p\u003e\n\u003cul\u003e\n\u003cli\u003e\n\u003cp\u003eSimilarity Filtering (Tanimoto):\nInfo: filters out the samples, which are not higher than the given similarity treshhold. Similarity is calculated with the Tanimoto distance of the targets and new made molecules.\nInput:\u003cbr\u003e\ncutoff_fingerprint_method = \"Morgan\"\ncutoff_similarity_method = \"Tanimoto\"\nsimilarity_threshold = [0, 0.9]\nOutput:  \u003c/p\u003e\n\u003c/li\u003e\n\u003cli\u003e\n\u003cp\u003eSimilarity Sampling Filter:\nInfo: samples a specific amount of a given samplespace (made by the simlarity filter). The similarity filter has to be activated!!!\nInput:\ncutoff_fingerprint_method = \"Morgan\"\ncutoff_similarity_method = \"Tanimoto\"\nsimilarity_threshold = [0, 0.9]\nOutput:\nn-samples\u003c/p\u003e\n\u003c/li\u003e\n\u003cli\u003e\n\u003cp\u003eMetabolomics filter:\nInput: list of experimental mass-to-charge (m/z), list of possible adducts\nOutput: compares the two lists by the m/z values and give back all compounds with a specific treshhold\u003c/p\u003e\n\u003c/li\u003e\n\u003cli\u003e\n\u003cp\u003eThermodynamic filter:\nInfo: eQuilator (module) biochemical thermodynamic calculator\nInput: \u003c/p\u003e\n\u003c/li\u003e\n\u003cli\u003e\n\u003cp\u003eFeasibility filter:\nInfo: DeepRFC (\u003ca href=\"https://%E2%80%8Bdoi.%E2%80%8Borg/%E2%80%8B10.%E2%80%8B1002/%E2%80%8Bbiot.%E2%80%8B20200%E2%80%8B0605\"\u003ehttps://​doi.​org/​10.​1002/​biot.​20200​0605\u003c/a\u003e) DeepRFC utilizes substrate-product pairs within KEGG reactions to train a deep neural net to predict a reaction’s feasibility.\u003c/p\u003e\n\u003c/li\u003e\n\u003c/ul\u003e\n\u003cp\u003e\u003cimg src=\"/commons-dws-public/expanded-np-chemspace.distancemeters_chemical_moleculs.png\" alt=\" distancemeters \"\u003e\u003cbr\u003e\n\u003cem\u003edistancemeters \u003ca href=\"https://www.sciencedirect.com/science/article/pii/S1046202314002631?via%3Dihub\"\u003ehttps://www.sciencedirect.com/science/article/pii/S1046202314002631?via%3Dihub\u003c/a\u003e\u003c/em\u003e\u003c/p\u003e\n\u003ch3 id=\"a-deep-learning-approach-to-evaluate-the-feasibility-of-enzymatic-reactions-generated-by-retrobiosynthesis\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#a-deep-learning-approach-to-evaluate-the-feasibility-of-enzymatic-reactions-generated-by-retrobiosynthesis\"\u003e\u003c/a\u003e\u003ca href=\"https://analyticalsciencejournals.onlinelibrary.wiley.com/doi/10.1002/biot.202000605\"\u003eA deep learning approach to evaluate the feasibility of enzymatic reactions generated by retrobiosynthesis\u003c/a\u003e\u003c/h3\u003e\n\u003cp\u003eenzymatic reactions generated by retrobiosynthesis\nDOI: 10.1002/biot.202000605\u003c/p\u003e\n\u003ch3 id=\"rdp---filter\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#rdp---filter\"\u003e\u003c/a\u003e\u003ca href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4920114/\"\u003eRDP - Filter\u003c/a\u003e\u003c/h3\u003e\n\u003cp\u003e10.1093/bioinformatics/btw096\u003c/p\u003e\n\u003cp\u003e\u003ca href=\"/commons-dws-public/notes/3ca85tg6eqf78kgql1mj3vl\"\u003eAbbreviations\u003c/a\u003e\u003c/p\u003e\n\u003chr\u003e\n\u003cstrong\u003eBacklinks\u003c/strong\u003e\n\u003cul\u003e\n\u003cli\u003e\u003ca href=\"/commons-dws-public/notes/0vbuzuauxdhmsbckhc0pye0\"\u003eInformation\u003c/a\u003e\u003c/li\u003e\n\u003cli\u003e\u003ca href=\"/commons-dws-public/notes/qy0iqvyrj0me4co8u8d25nh\"\u003e2024-02-21\u003c/a\u003e\u003c/li\u003e\n\u003cli\u003e\u003ca href=\"/commons-dws-public/notes/oxoker8doqq8gxa0e16wnp1\"\u003e2024-02-22\u003c/a\u003e\u003c/li\u003e\n\u003cli\u003e\u003ca href=\"/commons-dws-public/notes/wn10dc3l535y60u8yb9iq2d\"\u003e2024-02-23\u003c/a\u003e\u003c/li\u003e\n\u003c/ul\u003e","noteIndex":{"id":"QvYK9hGbCvPpEfSRYhV8j","title":"Welcome to the COMMONS Lab Open Dendron","desc":"","updated":1693056226815,"created":1630130450048,"custom":{"nav_order":0,"permalink":"/"},"fname":"root","type":"note","vault":{"fsPath":"vault"},"contentHash":"20f67df3dd870fc450a162c202c9ff41","links":[{"type":"wiki","from":{"fname":"root","id":"QvYK9hGbCvPpEfSRYhV8j","vaultName":"vault"},"value":"open-notebook.commons.setup","position":{"start":{"line":43,"column":118,"offset":1871},"end":{"line":43,"column":149,"offset":1902},"indent":[]},"xvault":false,"sameFile":false,"to":{"fname":"open-notebook.commons.setup"}}],"anchors":{"what-is-this-note":{"type":"header","text":"What is this note","value":"what-is-this-note","line":16,"column":0,"depth":2},"tutorial":{"type":"header","text":"Tutorial","value":"tutorial","line":20,"column":0,"depth":2},"dendron":{"type":"header","text":"Dendron.","value":"dendron","line":22,"column":0,"depth":3},"what-is-dendron-":{"type":"header","text":"What is Dendron ?","value":"what-is-dendron-","line":24,"column":0,"depth":4},"where-to-get-more-info-on-dendron-":{"type":"header","text":"Where to get more info on Dendron ?","value":"where-to-get-more-info-on-dendron-","line":30,"column":0,"depth":4},"concrete-steps-to-access-and-contribute-to-the-dbgi-dendron":{"type":"header","text":"Concrete steps to access and contribute to the DBGI Dendron","value":"concrete-steps-to-access-and-contribute-to-the-dbgi-dendron","line":35,"column":0,"depth":4},"questions-comments-suggestions-":{"type":"header","text":"Questions, comments, suggestions ?","value":"questions-comments-suggestions-","line":52,"column":0,"depth":2}},"children":["5o0tvzf4l6t6moau7en1v48","e4ul30admmxm9ilxfq275cq","h78qbvh1fd9bwcykmcm9uo3","6ooxuf27zzf3grb14o18qf2","tiqo5upbg6224t7i8p87iok","ltq8wcrabmq6mlescrdi4m0","bdq3suiz8yriznljea2zpug","ih6vyup3yre4m9woc9ldl7k","vDTgZL9UHWqYtBFdtD3vK","69ekb2qhuwukr0jwrzy6tnw","8us579vydjfskegndhvdzsl","373694o1o6ocqohko4um7c9","jldFUSJGjDf1mFH8c2yUI","dh2qp7w4tl6otf89xrmxitv","2rvdf4t5qnx5hqjj29bojcc","stj9lq0lxhfuk5ntomiozz0","gvdyfgvq0z08fcfaqzmhtjo","2rvclk21kfedpvxuzjpizi7","dcwgamgyghrlyau1avlh6ug","432qdutuo1i8h5pcuh39ytt","2npxodkvs5jk6p1eksi551b","h81rw16zzgh9kix8ti1mbam","lco50o42dfeph5i5f2k981w","46d1odkovrpdi6g16992phy","sw67lr6069gwhqmdym2l9na","s9gzk7nphtjzmpdqvk5y4vj","82ra6p4ykf814hp8yfw4wp6","hpnaglyhnb48vuiniqecnqf","y7nk7xmiht44neo0q7cl5cs","2cxr093kvrg4jojokwgqrbj","8vfu5h3617jbzy78xr2hlo1","w75krbmkkvla3hwd66hn5bh","40po5od17ekcr1e77dluxao","4pqppog2u66rx1tkc8jy5v9","ektni5nfulyro7dkrbtek7j","vfsp2aci6vy7kg2jsyy8tzu","0wm4geq40t6nscn88rcwu47","ib4jmxy1x3l1r0ewe6hln4j","0h57aw0be2jnc3dxcpn3ajn","4789z8f1dqdh5k3nmgr1m4z","p5dszyuhj1klgje0brtv0xj","sk4m9gq7vz2t9phcxlxvqpw","43p60vv8btyhaclugl93jok","zhc9hhpaobkbo4e2votv6wr","ylulfzi7ra5yv6ja7j78t6e","wj0i9czxvt4v0b674nb8501","ovslc641d0h00s16tf6kl8w","4zmlbmcp6q4q69ycvhavzr6","59ftsVxFXXM9bB43vfKwK","nmnpsy2e1rwqnqgurogi2u6","yi9sq691cas9eyjfx8x7atd","hmn2egle6t4foxe01xup7xt","3yysbx5aor30qd48lqaulbt","nq0iph4a21h5alnuip05j6z","y4t7h534eugai0bjr3mgfjn","zlbfl274tfdx3ix844jt5bm","ulasj276ktykjhdco456wan","au21tj497d981l8pdsily19","8zdzqjzai32lw7xbn30khlt","rjq88ar936t5kn7i86dnfb2","q1yxhdq1jqazej8hq8qzqrb","9mwhjkgi6lx9icrry5qwcg7","3j65st8vcb7mztobso4xk1m","t4z6dnhqmj5axvc2d1esbnn","quvftc4ygczckg0vfltkxbb","jka9lkt701hkbxyfs0qujnx","ivro3wr1xttk7fh7ikp0mvx","alwx9pow583s15xgx2gecjv"],"parent":null,"data":{},"body":"\n![](/assets/images/allard-lab.jpg)\n\nWelcome the the COMMONS Lab Open Dendron !\n\nIn the [COMMONS Lab](https://www.unifr.ch/bio/en/groups/allard/) we intent to follow the [Open Notebook Science](https://en.wikipedia.org/wiki/Open-notebook_science) approach to document our research.\n\nFor this we use the [Dendron](https://www.dendron.so/) system as a mean to efficiently capture notes and publish them.\n\n## What is this note\n\nThis note is a succinct tutorial note aiming to get you started in the use of the COMMONS Lab Dendron.\n\n## Tutorial\n\n### Dendron. \n\n#### What is Dendron ?\n\n\u003e Dendron is an open-source, local-first, markdown-based, note-taking tool. Think of it as a cache for everything that you care about - if you've spent more then five minutes solving a problem, you should never spent any more time solving the same exact problem.\n\u003e \n\u003e Dendron is a knowledge base built by and for developers and integrates natively with IDEs like VS Code and VSCodium.\n\n#### Where to get more info on Dendron ?\n\n- You can get more information in the Dendron system at the official website www.dendron.so\n- All the Dendron documentation is hosted here https://wiki.dendron.so/. It is, obviously, a Dendron itself.\n\n#### Concrete steps to access and contribute to the DBGI Dendron\n\n1. Install [VSCode](https://code.visualstudio.com/download)\n2. Install [Dendron](https://marketplace.visualstudio.com/items?itemName=dendron.dendron) from the VSCode marketplace \n3. Clone the COMMONS Dendron repo\n\n    In your terminal\n    ```\n    git clone https://github.com/commons-research/commons-dws-public.git\n    ```\n4. In VSCode open (File/Open) the cloned repository.\n5. Voila ! You should now be in the COMMONS Lab Open Dendron.\n\nFeel free to explore and contribute.\nA first, important step could be to configure VSCode so that you can easily add a daily note. See steps described at [[open-notebook.commons.setup]]\n\n\n## Questions, comments, suggestions ?\n\nFeel free to contribute here https://github.com/orgs/commons-research/discussions\n\n"},"collectionChildren":null,"customHeadContent":null,"config":{"version":5,"dev":{"enablePreviewV2":true},"commands":{"lookup":{"note":{"selectionMode":"extract","confirmVaultOnCreate":false,"leaveTrace":false,"bubbleUpCreateNew":true,"vaultSelectionModeOnCreate":"smart","fuzzThreshold":0.2}},"insertNote":{"initialValue":"templates"},"insertNoteLink":{"aliasMode":"none","enableMultiSelect":false},"insertNoteIndex":{"enableMarker":false},"randomNote":{},"copyNoteLink":{"aliasMode":"title"},"templateHierarchy":"template"},"workspace":{"vaults":[{"fsPath":"vault"}],"journal":{"dailyDomain":"daily","name":"journal","dateFormat":"y.MM.dd","addBehavior":"childOfDomain"},"scratch":{"name":"scratch","dateFormat":"y.MM.dd.HHmmss","addBehavior":"asOwnDomain"},"graph":{"zoomSpeed":1,"createStub":false},"enableAutoCreateOnDefinition":false,"enableXVaultWikiLink":false,"enableRemoteVaultInit":true,"workspaceVaultSyncMode":"noCommit","enableAutoFoldFrontmatter":false,"maxPreviewsCached":10,"maxNoteLength":204800,"task":{"name":"","dateFormat":"","addBehavior":"childOfCurrent","statusSymbols":{"":" ","wip":"w","done":"x","assigned":"a","moved":"m","blocked":"b","delegated":"l","dropped":"d","pending":"y"},"prioritySymbols":{"H":"high","M":"medium","L":"low"},"todoIntegration":false,"createTaskSelectionType":"selection2link","taskCompleteStatus":["done","x"]},"enableUserTags":true,"enableHashTags":true,"dendronVersion":"0.112.0","enableEditorDecorations":true,"enableFullHierarchyNoteTitle":false},"preview":{"enableFMTitle":true,"enableNoteTitleForLink":true,"enableMermaid":true,"enablePrettyRefs":true,"enableKatex":true,"automaticallyShowPreview":false,"enableFrontmatterTags":true,"enableHashesForFMTags":false},"publishing":{"enableFMTitle":true,"enableNoteTitleForLink":true,"enableMermaid":true,"enablePrettyRefs":true,"enableKatex":true,"assetsPrefix":"/commons-dws-public","copyAssets":true,"siteHierarchies":["root"],"enableSiteLastModified":true,"siteRootDir":"docs","siteUrl":"https://commons-research.github.io","enableFrontmatterTags":true,"enableHashesForFMTags":false,"enableRandomlyColoredTags":true,"duplicateNoteBehavior":{"action":"useVault","payload":["vault"]},"writeStubs":false,"seo":{"title":"COMMONS DWS Public","description":"Personal knowledge space"},"github":{"enableEditLink":true,"editLinkText":"Click here to edit this page on Github !","editBranch":"main","editViewMode":"edit","editRepository":"https://github.com/commons-research/commons-dws-public"},"enablePrettyLinks":true,"enableTaskNotes":true,"siteFaviconPath":"favicon.ico","siteIndex":"root","searchMode":"lookup"}}},"__N_SSG":true},"page":"/notes/[id]","query":{"id":"loztevovn8nanzdk302b756"},"buildId":"ilh4ComeLwDbrM8zcKyPY","assetPrefix":"/commons-dws-public","isFallback":false,"gsp":true,"scriptLoader":[]}</script></body></html>